[2022-01-24 13:38:12,375 WARNING ThreadPoolExecutor-0_0 base.py run_job 120] [38344] [Run time of job "ExecuteTestCaseController.execute_testsuite (trigger: interval[1:00:00], next run at: 2022-01-24 14:25:31 CST)" was missed by 0:12:40.515021]
[2022-01-24 13:38:12,405 WARNING ThreadPoolExecutor-0_3 base.py run_job 120] [47192] [Run time of job "ExecuteTestCaseController.execute_testsuite (trigger: interval[1:00:00], next run at: 2022-01-01 13:25:42 CST)" was missed by 0:12:30.367521]
[2022-01-24 13:38:12,413 WARNING ThreadPoolExecutor-0_3 base.py run_job 120] [41924] [Run time of job "ExecuteTestCaseController.execute_testsuite (trigger: interval[1:00:00], next run at: 2022-01-01 13:25:42 CST)" was missed by 0:12:30.375501]
[2022-01-01 15:10:39,819 INFO MainThread _internal.py _log 225] [30132] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:10:39,963 INFO MainThread _internal.py _log 225] [27856] [ * Restarting with stat]
[2022-01-01 15:10:44,772 INFO MainThread cron_utils.py start_cron 213] [50628] [开始定时任务]
[2022-01-01 15:10:44,856 INFO MainThread base.py start 166] [50628] [Scheduler started]
[2022-01-01 15:10:44,893 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:10:47,405 INFO MainThread cron_utils.py start_cron 213] [49496] [开始定时任务]
[2022-01-01 15:10:47,508 INFO MainThread base.py start 166] [49496] [Scheduler started]
[2022-01-01 15:10:47,533 WARNING MainThread _internal.py _log 225] [49496] [ * Debugger is active!]
[2022-01-01 15:10:47,558 INFO MainThread _internal.py _log 225] [49496] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:10:47,791 INFO Thread-1 _internal.py _log 225] [48132] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:13:35,793 INFO MainThread _internal.py _log 225] [49496] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:13:35,969 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:13:40,041 INFO MainThread cron_utils.py start_cron 213] [39792] [开始定时任务]
[2022-01-01 15:13:40,129 INFO MainThread base.py start 166] [39792] [Scheduler started]
[2022-01-01 15:13:40,172 WARNING MainThread _internal.py _log 225] [39792] [ * Debugger is active!]
[2022-01-01 15:13:40,209 INFO MainThread _internal.py _log 225] [39792] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:13:40,648 INFO Thread-1 _internal.py _log 225] [24240] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:14:35,203 INFO MainThread _internal.py _log 225] [39792] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:14:35,310 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:14:38,219 INFO MainThread cron_utils.py start_cron 213] [23072] [开始定时任务]
[2022-01-01 15:14:38,305 INFO MainThread base.py start 166] [23072] [Scheduler started]
[2022-01-01 15:14:38,335 WARNING MainThread _internal.py _log 225] [23072] [ * Debugger is active!]
[2022-01-01 15:14:38,362 INFO MainThread _internal.py _log 225] [23072] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:14:38,635 INFO Thread-1 _internal.py _log 225] [50020] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:17:13,874 INFO MainThread _internal.py _log 225] [23072] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:17:14,197 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:17:19,741 INFO MainThread cron_utils.py start_cron 213] [27136] [开始定时任务]
[2022-01-01 15:17:19,843 INFO MainThread base.py start 166] [27136] [Scheduler started]
[2022-01-01 15:17:19,887 WARNING MainThread _internal.py _log 225] [27136] [ * Debugger is active!]
[2022-01-01 15:17:19,928 INFO MainThread _internal.py _log 225] [27136] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:17:20,310 INFO Thread-1 _internal.py _log 225] [40256] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:17:25,017 INFO Thread-2 test_case_controller.py import_case 286] [42284] [从文件中读取的要导入的数据为：[{'id': 4.0, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:08:01', 'update_time': 'None', 'operation': 'None'}, {'id': 5.0, 'case_name': '用例5', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "{{phone}}", "password": "{{password}}"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:11:42', 'update_time': 'None', 'operation': 'None'}, {'id': 6.0, 'case_name': '用例6', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "13800138006", "password": "123456"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 15:09:40', 'update_time': 'None', 'operation': 'None'}]]
[2022-01-01 15:17:25,076 INFO Thread-2 __init__.py framework_error 73] [42284] [(pymysql.err.IntegrityError) (1062, "Duplicate entry '4' for key 'PRIMARY'")
[SQL: INSERT INTO test_case (id, case_name, method, protocal, host, port, path, params, headers, body, description, predict, suite_id, `isDeleted`, status, create_time, update_time, operation) VALUES (%(id)s, %(case_name)s, %(method)s, %(protocal)s, %(host)s, %(port)s, %(path)s, %(params)s, %(headers)s, %(body)s, %(description)s, %(predict)s, %(suite_id)s, %(isDeleted)s, %(status)s, %(create_time)s, %(update_time)s, %(operation)s)]
[parameters: {'id': 4.0, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:08:01', 'update_time': None, 'operation': None}]
(Background on this error at: https://sqlalche.me/e/14/gkpj)]
[2022-01-01 15:17:25,086 INFO Thread-2 _internal.py _log 225] [42284] [127.0.0.1 - - [01/Jan/2022 15:17:25] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 15:17:54,018 INFO Thread-3 test_case_controller.py import_case 286] [3400] [从文件中读取的要导入的数据为：[{'id': 4.0, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:08:01', 'update_time': 'None', 'operation': 'None'}, {'id': 5.0, 'case_name': '用例5', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "{{phone}}", "password": "{{password}}"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:11:42', 'update_time': 'None', 'operation': 'None'}, {'id': 6.0, 'case_name': '用例6', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "13800138006", "password": "123456"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 15:09:40', 'update_time': 'None', 'operation': 'None'}]]
[2022-01-01 15:17:54,261 INFO Thread-3 test_case_controller.py import_case 297] [3400] [返回响应数据为：[{'id': 4.0, 'case_name': '用例4'}, {'id': 5.0, 'case_name': '用例5'}, {'id': 6.0, 'case_name': '用例6'}]]
[2022-01-01 15:17:54,262 INFO Thread-3 _internal.py _log 225] [3400] [127.0.0.1 - - [01/Jan/2022 15:17:54] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 15:18:42,837 INFO MainThread _internal.py _log 225] [27136] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:18:43,003 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:18:48,764 INFO MainThread cron_utils.py start_cron 213] [47612] [开始定时任务]
[2022-01-01 15:18:48,846 INFO MainThread base.py start 166] [47612] [Scheduler started]
[2022-01-01 15:18:48,900 WARNING MainThread _internal.py _log 225] [47612] [ * Debugger is active!]
[2022-01-01 15:18:48,946 INFO MainThread _internal.py _log 225] [47612] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:18:49,373 INFO Thread-1 _internal.py _log 225] [41216] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:19:06,186 INFO Thread-2 test_case_controller.py import_case 286] [39884] [从文件中读取的要导入的数据为：[{'id': 4.0, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:08:01', 'update_time': 'None', 'operation': 'None'}, {'id': 5.0, 'case_name': '用例5', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "{{phone}}", "password": "{{password}}"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:11:42', 'update_time': 'None', 'operation': 'None'}, {'id': 6.0, 'case_name': '用例6', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "13800138006", "password": "123456"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 15:09:40', 'update_time': 'None', 'operation': 'None'}]]
[2022-01-01 15:19:06,338 INFO Thread-2 test_case_controller.py import_case 297] [39884] [返回响应数据为：[{'id': 4.0, 'case_name': '用例4'}, {'id': 5.0, 'case_name': '用例5'}, {'id': 6.0, 'case_name': '用例6'}]]
[2022-01-01 15:19:06,340 INFO Thread-2 _internal.py _log 225] [39884] [127.0.0.1 - - [01/Jan/2022 15:19:06] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 15:19:31,872 INFO MainThread _internal.py _log 225] [47612] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:19:32,081 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:19:36,415 INFO MainThread cron_utils.py start_cron 213] [51144] [开始定时任务]
[2022-01-01 15:19:36,500 INFO MainThread base.py start 166] [51144] [Scheduler started]
[2022-01-01 15:19:36,545 WARNING MainThread _internal.py _log 225] [51144] [ * Debugger is active!]
[2022-01-01 15:19:36,583 INFO MainThread _internal.py _log 225] [51144] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:19:36,983 INFO Thread-1 _internal.py _log 225] [3780] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:19:47,642 INFO Thread-2 test_case_controller.py import_case 286] [33608] [从文件中读取的要导入的数据为：[{'id': 4.0, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:08:01', 'update_time': 'None', 'operation': 'None'}, {'id': 5.0, 'case_name': '用例5', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "{{phone}}", "password": "{{password}}"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 14:11:42', 'update_time': 'None', 'operation': 'None'}, {'id': 6.0, 'case_name': '用例6', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': '8080', 'path': '/ssm_web/user/login', 'params': '{"phone": "13800138006", "password": "123456"}', 'headers': '{"Content-Type": "application/json"}', 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '{"assert_response_code": {"response_code": 200}, "assert_json": {"path": "$.state", "predict_value": 206}, "assert_time": {"predict_time": 1}, "assert_reg": {"reg": "{\'success\':true,\'state\':(.+?),", "predict_value": "{{state}}"}}', 'suite_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-31 15:09:40', 'update_time': 'None', 'operation': 'None'}]]
[2022-01-01 15:19:47,817 INFO Thread-2 test_case_controller.py import_case 297] [33608] [返回响应数据为：[{'id': 4, 'case_name': '用例4'}, {'id': 5, 'case_name': '用例5'}, {'id': 6, 'case_name': '用例6'}]]
[2022-01-01 15:19:47,819 INFO Thread-2 _internal.py _log 225] [33608] [127.0.0.1 - - [01/Jan/2022 15:19:47] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 15:55:30,813 INFO MainThread _internal.py _log 225] [51144] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:55:30,994 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:55:35,881 INFO MainThread cron_utils.py start_cron 213] [13092] [开始定时任务]
[2022-01-01 15:55:35,997 INFO MainThread base.py start 166] [13092] [Scheduler started]
[2022-01-01 15:55:36,050 WARNING MainThread _internal.py _log 225] [13092] [ * Debugger is active!]
[2022-01-01 15:55:36,098 INFO MainThread _internal.py _log 225] [13092] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:55:37,079 INFO Thread-1 _internal.py _log 225] [38016] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:56:49,536 INFO MainThread _internal.py _log 225] [13092] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:56:49,716 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:56:54,698 INFO MainThread cron_utils.py start_cron 213] [50444] [开始定时任务]
[2022-01-01 15:56:54,782 INFO MainThread base.py start 166] [50444] [Scheduler started]
[2022-01-01 15:56:54,830 WARNING MainThread _internal.py _log 225] [50444] [ * Debugger is active!]
[2022-01-01 15:56:54,871 INFO MainThread _internal.py _log 225] [50444] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:56:55,363 INFO Thread-1 _internal.py _log 225] [50884] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:57:27,888 INFO MainThread _internal.py _log 225] [50444] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:57:28,194 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:57:32,558 INFO MainThread cron_utils.py start_cron 213] [48732] [开始定时任务]
[2022-01-01 15:57:32,660 INFO MainThread base.py start 166] [48732] [Scheduler started]
[2022-01-01 15:57:32,705 WARNING MainThread _internal.py _log 225] [48732] [ * Debugger is active!]
[2022-01-01 15:57:32,753 INFO MainThread _internal.py _log 225] [48732] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:57:33,204 INFO Thread-1 _internal.py _log 225] [50944] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:57:47,654 INFO MainThread _internal.py _log 225] [48732] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:57:47,830 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 15:57:52,822 INFO MainThread cron_utils.py start_cron 213] [50156] [开始定时任务]
[2022-01-01 15:57:52,919 INFO MainThread base.py start 166] [50156] [Scheduler started]
[2022-01-01 15:57:52,964 WARNING MainThread _internal.py _log 225] [50156] [ * Debugger is active!]
[2022-01-01 15:57:53,003 INFO MainThread _internal.py _log 225] [50156] [ * Debugger PIN: 133-332-853]
[2022-01-01 15:57:53,483 INFO Thread-1 _internal.py _log 225] [37536] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 15:58:30,670 INFO MainThread _internal.py _log 225] [50156] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 15:58:30,828 INFO MainThread _internal.py _log 225] [50628] [ * Restarting with stat]
[2022-01-01 16:12:57,203 INFO MainThread cron_utils.py start_cron 213] [45176] [开始定时任务]
[2022-01-01 16:12:57,312 INFO MainThread base.py start 166] [45176] [Scheduler started]
[2022-01-01 16:12:57,357 INFO MainThread _internal.py _log 225] [45176] [ * Restarting with stat]
[2022-01-01 16:13:01,266 INFO MainThread cron_utils.py start_cron 213] [46004] [开始定时任务]
[2022-01-01 16:13:01,353 INFO MainThread base.py start 166] [46004] [Scheduler started]
[2022-01-01 16:13:01,392 WARNING MainThread _internal.py _log 225] [46004] [ * Debugger is active!]
[2022-01-01 16:13:01,430 INFO MainThread _internal.py _log 225] [46004] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:13:01,792 INFO Thread-1 _internal.py _log 225] [49212] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:13:49,926 INFO Thread-2 test_case_controller.py patch 468] [47252] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '1'}]
[2022-01-01 16:13:49,981 INFO Thread-2 _internal.py _log 225] [47252] [127.0.0.1 - - [01/Jan/2022 16:13:49] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:14:33,062 INFO Thread-3 test_case_controller.py patch 468] [47388] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:14:33,118 INFO Thread-3 __init__.py framework_error 73] [47388] [Instance <TestCaseModel at 0x1704b904b38> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)]
[2022-01-01 16:14:33,119 INFO Thread-3 _internal.py _log 225] [47388] [127.0.0.1 - - [01/Jan/2022 16:14:33] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:16:11,522 INFO MainThread _internal.py _log 225] [46004] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 16:16:11,686 INFO MainThread _internal.py _log 225] [45176] [ * Restarting with stat]
[2022-01-01 16:16:16,751 INFO MainThread cron_utils.py start_cron 213] [46836] [开始定时任务]
[2022-01-01 16:16:16,833 INFO MainThread base.py start 166] [46836] [Scheduler started]
[2022-01-01 16:16:16,879 WARNING MainThread _internal.py _log 225] [46836] [ * Debugger is active!]
[2022-01-01 16:16:16,925 INFO MainThread _internal.py _log 225] [46836] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:16:17,330 INFO Thread-1 _internal.py _log 225] [9624] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:16:30,962 INFO MainThread _internal.py _log 225] [46836] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 16:16:31,138 INFO MainThread _internal.py _log 225] [45176] [ * Restarting with stat]
[2022-01-01 16:16:35,288 INFO MainThread cron_utils.py start_cron 213] [17608] [开始定时任务]
[2022-01-01 16:16:35,366 INFO MainThread base.py start 166] [17608] [Scheduler started]
[2022-01-01 16:16:35,409 WARNING MainThread _internal.py _log 225] [17608] [ * Debugger is active!]
[2022-01-01 16:16:35,451 INFO MainThread _internal.py _log 225] [17608] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:16:35,865 INFO Thread-1 _internal.py _log 225] [46032] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:16:41,005 INFO Thread-2 test_case_controller.py patch 471] [25896] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:16:41,054 INFO Thread-2 test_case_controller.py switch_status 305] [25896] [<TestCaseModel> '用例4']
[2022-01-01 16:16:41,054 INFO Thread-2 test_case_controller.py switch_status 319] [25896] [{'status': '1'}]
[2022-01-01 16:16:41,105 INFO Thread-2 __init__.py framework_error 73] [25896] [Instance <TestCaseModel at 0x1b2d471d2b0> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)]
[2022-01-01 16:16:41,106 INFO Thread-2 _internal.py _log 225] [25896] [127.0.0.1 - - [01/Jan/2022 16:16:41] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:17:20,305 INFO MainThread _internal.py _log 225] [17608] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 16:17:20,473 INFO MainThread _internal.py _log 225] [45176] [ * Restarting with stat]
[2022-01-01 16:17:25,783 INFO MainThread cron_utils.py start_cron 213] [49100] [开始定时任务]
[2022-01-01 16:17:25,871 INFO MainThread base.py start 166] [49100] [Scheduler started]
[2022-01-01 16:17:25,918 WARNING MainThread _internal.py _log 225] [49100] [ * Debugger is active!]
[2022-01-01 16:17:25,963 INFO MainThread _internal.py _log 225] [49100] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:17:26,381 INFO Thread-1 _internal.py _log 225] [49208] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:17:32,362 INFO MainThread cron_utils.py start_cron 213] [45980] [开始定时任务]
[2022-01-01 16:17:32,766 INFO MainThread base.py start 166] [45980] [Scheduler started]
[2022-01-01 16:17:32,808 INFO MainThread _internal.py _log 225] [45980] [ * Restarting with stat]
[2022-01-01 16:17:37,210 INFO MainThread cron_utils.py start_cron 213] [48996] [开始定时任务]
[2022-01-01 16:17:37,294 INFO MainThread base.py start 166] [48996] [Scheduler started]
[2022-01-01 16:17:37,340 WARNING MainThread _internal.py _log 225] [48996] [ * Debugger is active!]
[2022-01-01 16:17:37,381 INFO MainThread _internal.py _log 225] [48996] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:17:37,855 INFO Thread-1 _internal.py _log 225] [41272] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:17:38,672 INFO Thread-2 test_case_controller.py patch 472] [50872] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:17:38,719 INFO Thread-2 test_case_controller.py switch_status 305] [50872] [<TestCaseModel> '用例4']
[2022-01-01 16:17:38,720 INFO Thread-2 test_case_controller.py switch_status 319] [50872] [{'status': '0'}]
[2022-01-01 16:17:38,758 INFO Thread-2 __init__.py framework_error 73] [50872] [Instance <TestCaseModel at 0x21fbde5e2e8> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)]
[2022-01-01 16:17:38,760 INFO Thread-2 _internal.py _log 225] [50872] [127.0.0.1 - - [01/Jan/2022 16:17:38] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:17:57,877 INFO Thread-3 test_case_controller.py patch 472] [50324] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:17:57,887 INFO Thread-3 test_case_controller.py switch_status 305] [50324] [<TestCaseModel> '用例4']
[2022-01-01 16:17:57,887 INFO Thread-3 test_case_controller.py switch_status 319] [50324] [{'status': '1'}]
[2022-01-01 16:17:57,928 INFO Thread-3 __init__.py framework_error 73] [50324] [Instance <TestCaseModel at 0x21fbde5eeb8> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: https://sqlalche.me/e/14/bhk3)]
[2022-01-01 16:17:57,929 INFO Thread-3 _internal.py _log 225] [50324] [127.0.0.1 - - [01/Jan/2022 16:17:57] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:19:10,943 INFO MainThread _internal.py _log 225] [48996] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_case_controller.py', reloading]
[2022-01-01 16:19:11,225 INFO MainThread _internal.py _log 225] [45980] [ * Restarting with stat]
[2022-01-01 16:19:16,035 INFO MainThread cron_utils.py start_cron 213] [28576] [开始定时任务]
[2022-01-01 16:19:16,133 INFO MainThread base.py start 166] [28576] [Scheduler started]
[2022-01-01 16:19:16,179 WARNING MainThread _internal.py _log 225] [28576] [ * Debugger is active!]
[2022-01-01 16:19:16,224 INFO MainThread _internal.py _log 225] [28576] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:19:16,643 INFO Thread-1 _internal.py _log 225] [39308] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:19:25,679 INFO MainThread cron_utils.py start_cron 213] [43072] [开始定时任务]
[2022-01-01 16:19:25,774 INFO MainThread base.py start 166] [43072] [Scheduler started]
[2022-01-01 16:19:25,820 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:19:29,985 INFO MainThread cron_utils.py start_cron 213] [33868] [开始定时任务]
[2022-01-01 16:19:30,077 INFO MainThread base.py start 166] [33868] [Scheduler started]
[2022-01-01 16:19:30,133 WARNING MainThread _internal.py _log 225] [33868] [ * Debugger is active!]
[2022-01-01 16:19:30,180 INFO MainThread _internal.py _log 225] [33868] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:19:30,723 INFO Thread-1 _internal.py _log 225] [48008] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:19:31,110 INFO Thread-2 test_case_controller.py patch 472] [44592] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:19:31,151 INFO Thread-2 test_case_controller.py switch_status 305] [44592] [<TestCaseModel> '用例4']
[2022-01-01 16:19:31,151 INFO Thread-2 test_case_controller.py switch_status 319] [44592] [{'status': '0'}]
[2022-01-01 16:19:31,212 INFO Thread-2 test_case_controller.py switch_status 329] [44592] [{'id': '4', 'status': '0'}]
[2022-01-01 16:19:31,213 INFO Thread-2 _internal.py _log 225] [44592] [127.0.0.1 - - [01/Jan/2022 16:19:31] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:19:36,392 INFO Thread-3 test_case_controller.py patch 472] [51000] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:19:36,410 INFO Thread-3 test_case_controller.py switch_status 305] [51000] [<TestCaseModel> '用例4']
[2022-01-01 16:19:36,411 INFO Thread-3 test_case_controller.py switch_status 319] [51000] [{'status': '1'}]
[2022-01-01 16:19:36,497 INFO Thread-3 test_case_controller.py switch_status 329] [51000] [{'id': '4', 'status': '1'}]
[2022-01-01 16:19:36,499 INFO Thread-3 _internal.py _log 225] [51000] [127.0.0.1 - - [01/Jan/2022 16:19:36] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:19:38,652 INFO Thread-4 test_case_controller.py patch 472] [49140] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:19:38,665 INFO Thread-4 test_case_controller.py switch_status 305] [49140] [<TestCaseModel> '用例4']
[2022-01-01 16:19:38,666 INFO Thread-4 test_case_controller.py switch_status 319] [49140] [{'status': '0'}]
[2022-01-01 16:19:38,693 INFO Thread-4 test_case_controller.py switch_status 329] [49140] [{'id': '4', 'status': '0'}]
[2022-01-01 16:19:38,694 INFO Thread-4 _internal.py _log 225] [49140] [127.0.0.1 - - [01/Jan/2022 16:19:38] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:19:39,685 INFO Thread-5 test_case_controller.py patch 472] [50944] [传入的禁用启用的用例为：{'project_id': '1', 'suite_id': '1', 'id': '4'}]
[2022-01-01 16:19:39,701 INFO Thread-5 test_case_controller.py switch_status 305] [50944] [<TestCaseModel> '用例4']
[2022-01-01 16:19:39,701 INFO Thread-5 test_case_controller.py switch_status 319] [50944] [{'status': '1'}]
[2022-01-01 16:19:39,729 INFO Thread-5 test_case_controller.py switch_status 329] [50944] [{'id': '4', 'status': '1'}]
[2022-01-01 16:19:39,730 INFO Thread-5 _internal.py _log 225] [50944] [127.0.0.1 - - [01/Jan/2022 16:19:39] "PATCH /api/testcase HTTP/1.1" 200 -]
[2022-01-01 16:29:36,708 INFO MainThread _internal.py _log 225] [33868] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:29:36,923 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:29:41,638 INFO MainThread cron_utils.py start_cron 213] [47344] [开始定时任务]
[2022-01-01 16:29:41,727 INFO MainThread base.py start 166] [47344] [Scheduler started]
[2022-01-01 16:29:41,772 WARNING MainThread _internal.py _log 225] [47344] [ * Debugger is active!]
[2022-01-01 16:29:41,812 INFO MainThread _internal.py _log 225] [47344] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:29:42,255 INFO Thread-1 _internal.py _log 225] [33732] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:30:35,925 INFO MainThread _internal.py _log 225] [47344] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:30:36,142 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:30:40,909 INFO MainThread cron_utils.py start_cron 213] [47076] [开始定时任务]
[2022-01-01 16:30:40,994 INFO MainThread base.py start 166] [47076] [Scheduler started]
[2022-01-01 16:30:41,044 WARNING MainThread _internal.py _log 225] [47076] [ * Debugger is active!]
[2022-01-01 16:30:41,091 INFO MainThread _internal.py _log 225] [47076] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:30:41,499 INFO Thread-1 _internal.py _log 225] [48380] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:31:34,148 INFO MainThread _internal.py _log 225] [47076] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:31:34,333 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:31:38,803 INFO MainThread cron_utils.py start_cron 213] [48552] [开始定时任务]
[2022-01-01 16:31:38,918 INFO MainThread base.py start 166] [48552] [Scheduler started]
[2022-01-01 16:31:38,985 WARNING MainThread _internal.py _log 225] [48552] [ * Debugger is active!]
[2022-01-01 16:31:39,038 INFO MainThread _internal.py _log 225] [48552] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:31:39,474 INFO Thread-1 _internal.py _log 225] [48316] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:31:55,459 INFO MainThread _internal.py _log 225] [48552] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:31:55,672 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:32:00,670 INFO MainThread cron_utils.py start_cron 213] [29500] [开始定时任务]
[2022-01-01 16:32:00,764 INFO MainThread base.py start 166] [29500] [Scheduler started]
[2022-01-01 16:32:00,831 WARNING MainThread _internal.py _log 225] [29500] [ * Debugger is active!]
[2022-01-01 16:32:00,884 INFO MainThread _internal.py _log 225] [29500] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:32:01,371 INFO Thread-1 _internal.py _log 225] [42204] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:32:24,400 INFO MainThread _internal.py _log 225] [29500] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:32:24,586 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:32:29,608 INFO MainThread cron_utils.py start_cron 213] [50592] [开始定时任务]
[2022-01-01 16:32:29,728 INFO MainThread base.py start 166] [50592] [Scheduler started]
[2022-01-01 16:32:29,787 WARNING MainThread _internal.py _log 225] [50592] [ * Debugger is active!]
[2022-01-01 16:32:29,835 INFO MainThread _internal.py _log 225] [50592] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:32:30,295 INFO Thread-1 _internal.py _log 225] [50860] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:32:53,502 INFO MainThread _internal.py _log 225] [50592] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:32:53,653 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:32:58,992 INFO MainThread cron_utils.py start_cron 213] [34260] [开始定时任务]
[2022-01-01 16:32:59,078 INFO MainThread base.py start 166] [34260] [Scheduler started]
[2022-01-01 16:32:59,132 WARNING MainThread _internal.py _log 225] [34260] [ * Debugger is active!]
[2022-01-01 16:32:59,183 INFO MainThread _internal.py _log 225] [34260] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:32:59,605 INFO Thread-1 _internal.py _log 225] [7908] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:33:18,325 INFO MainThread _internal.py _log 225] [34260] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:33:18,496 INFO MainThread _internal.py _log 225] [43072] [ * Restarting with stat]
[2022-01-01 16:33:23,313 INFO MainThread cron_utils.py start_cron 213] [36816] [开始定时任务]
[2022-01-01 16:33:23,428 INFO MainThread base.py start 166] [36816] [Scheduler started]
[2022-01-01 16:33:23,477 WARNING MainThread _internal.py _log 225] [36816] [ * Debugger is active!]
[2022-01-01 16:33:23,514 INFO MainThread _internal.py _log 225] [36816] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:33:23,901 INFO Thread-1 _internal.py _log 225] [2452] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:33:43,647 INFO MainThread cron_utils.py start_cron 213] [26616] [开始定时任务]
[2022-01-01 16:33:43,758 INFO MainThread base.py start 166] [26616] [Scheduler started]
[2022-01-01 16:33:43,809 INFO MainThread _internal.py _log 225] [26616] [ * Restarting with stat]
[2022-01-01 16:33:47,512 INFO MainThread cron_utils.py start_cron 213] [46244] [开始定时任务]
[2022-01-01 16:33:47,605 INFO MainThread base.py start 166] [46244] [Scheduler started]
[2022-01-01 16:33:47,645 WARNING MainThread _internal.py _log 225] [46244] [ * Debugger is active!]
[2022-01-01 16:33:47,685 INFO MainThread _internal.py _log 225] [46244] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:33:48,071 INFO Thread-1 _internal.py _log 225] [39180] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:34:51,326 INFO Thread-2 __init__.py abort_404 53] [36704] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 16:34:51,328 INFO Thread-2 _internal.py _log 225] [36704] [127.0.0.1 - - [01/Jan/2022 16:34:51] "POST /api/testsuite HTTP/1.1" 200 -]
[2022-01-01 16:35:52,623 INFO Thread-3 _internal.py _log 225] [3484] [127.0.0.1 - - [01/Jan/2022 16:35:52] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:36:23,273 INFO Thread-4 _internal.py _log 225] [41040] [127.0.0.1 - - [01/Jan/2022 16:36:23] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:36:29,611 INFO Thread-5 __init__.py abort_404 53] [36612] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 16:36:29,612 INFO Thread-5 _internal.py _log 225] [36612] [127.0.0.1 - - [01/Jan/2022 16:36:29] "POST /api/testsuite HTTP/1.1" 200 -]
[2022-01-01 16:36:36,009 INFO Thread-6 excel_utils.py export_data 67] [46936] [导出数据开始，准备导出的数据是：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]]
[2022-01-01 16:36:36,032 INFO Thread-6 _internal.py _log 225] [46936] [127.0.0.1 - - [01/Jan/2022 16:36:36] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:39:14,811 INFO MainThread _internal.py _log 225] [46244] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:39:14,964 INFO MainThread _internal.py _log 225] [26616] [ * Restarting with stat]
[2022-01-01 16:39:19,783 INFO MainThread cron_utils.py start_cron 213] [47220] [开始定时任务]
[2022-01-01 16:39:19,865 INFO MainThread base.py start 166] [47220] [Scheduler started]
[2022-01-01 16:39:19,921 WARNING MainThread _internal.py _log 225] [47220] [ * Debugger is active!]
[2022-01-01 16:39:19,971 INFO MainThread _internal.py _log 225] [47220] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:39:20,393 INFO Thread-1 _internal.py _log 225] [37368] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:39:59,410 INFO MainThread _internal.py _log 225] [47220] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:39:59,574 INFO MainThread _internal.py _log 225] [26616] [ * Restarting with stat]
[2022-01-01 16:40:04,539 INFO MainThread cron_utils.py start_cron 213] [46336] [开始定时任务]
[2022-01-01 16:40:04,654 INFO MainThread base.py start 166] [46336] [Scheduler started]
[2022-01-01 16:40:04,705 WARNING MainThread _internal.py _log 225] [46336] [ * Debugger is active!]
[2022-01-01 16:40:04,748 INFO MainThread _internal.py _log 225] [46336] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:40:05,167 INFO Thread-1 _internal.py _log 225] [29700] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:40:14,762 INFO MainThread cron_utils.py start_cron 213] [18720] [开始定时任务]
[2022-01-01 16:40:14,867 INFO MainThread base.py start 166] [18720] [Scheduler started]
[2022-01-01 16:40:14,913 INFO MainThread _internal.py _log 225] [18720] [ * Restarting with stat]
[2022-01-01 16:40:18,652 INFO MainThread cron_utils.py start_cron 213] [42660] [开始定时任务]
[2022-01-01 16:40:18,764 INFO MainThread base.py start 166] [42660] [Scheduler started]
[2022-01-01 16:40:18,824 WARNING MainThread _internal.py _log 225] [42660] [ * Debugger is active!]
[2022-01-01 16:40:18,884 INFO MainThread _internal.py _log 225] [42660] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:40:19,317 INFO Thread-1 _internal.py _log 225] [49728] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:41:55,484 INFO Thread-2 test_suite_controller.py import_case 145] [39064] [从文件中读取的要导入的数据为：[{'id': 3.0, 'suite_name': '支付', 'description': '描述内容', 'project_id': '1', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': 'None', 'operation': 'None'}]]
[2022-01-01 16:41:55,560 INFO Thread-2 test_suite_controller.py import_case 156] [39064] [返回响应数据为：[{'id': 3, 'suite_name': '支付'}]]
[2022-01-01 16:41:55,561 INFO Thread-2 _internal.py _log 225] [39064] [127.0.0.1 - - [01/Jan/2022 16:41:55] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:48:05,777 INFO MainThread _internal.py _log 225] [42660] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 16:48:05,947 INFO MainThread _internal.py _log 225] [18720] [ * Restarting with stat]
[2022-01-01 16:48:10,268 INFO MainThread cron_utils.py start_cron 213] [44936] [开始定时任务]
[2022-01-01 16:48:10,358 INFO MainThread base.py start 166] [44936] [Scheduler started]
[2022-01-01 16:48:10,408 WARNING MainThread _internal.py _log 225] [44936] [ * Debugger is active!]
[2022-01-01 16:48:10,449 INFO MainThread _internal.py _log 225] [44936] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:48:15,364 INFO MainThread cron_utils.py start_cron 213] [26664] [开始定时任务]
[2022-01-01 16:48:15,479 INFO MainThread base.py start 166] [26664] [Scheduler started]
[2022-01-01 16:48:15,525 INFO MainThread _internal.py _log 225] [26664] [ * Restarting with stat]
[2022-01-01 16:48:19,238 INFO MainThread cron_utils.py start_cron 213] [50044] [开始定时任务]
[2022-01-01 16:48:19,325 INFO MainThread base.py start 166] [50044] [Scheduler started]
[2022-01-01 16:48:19,369 WARNING MainThread _internal.py _log 225] [50044] [ * Debugger is active!]
[2022-01-01 16:48:19,411 INFO MainThread _internal.py _log 225] [50044] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:48:19,811 INFO Thread-1 _internal.py _log 225] [51192] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 16:49:13,498 INFO Thread-2 test_suite_controller.py patch 319] [47168] [传入的禁用启用的测试套件为：{'id': '1', 'project_id': 1}]
[2022-01-01 16:49:13,538 INFO Thread-2 test_suite_controller.py switch_status 164] [47168] [<TestSuiteModel> '登陆2']
[2022-01-01 16:49:13,539 INFO Thread-2 test_suite_controller.py switch_status 178] [47168] [{'status': '0'}]
[2022-01-01 16:49:13,574 INFO Thread-2 test_suite_controller.py switch_status 188] [47168] [{'id': '1', 'status': '0'}]
[2022-01-01 16:49:13,575 INFO Thread-2 _internal.py _log 225] [47168] [127.0.0.1 - - [01/Jan/2022 16:49:13] "PATCH /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:49:16,449 INFO Thread-3 test_suite_controller.py patch 319] [45048] [传入的禁用启用的测试套件为：{'id': '1', 'project_id': 1}]
[2022-01-01 16:49:16,461 INFO Thread-3 test_suite_controller.py switch_status 164] [45048] [<TestSuiteModel> '登陆2']
[2022-01-01 16:49:16,462 INFO Thread-3 test_suite_controller.py switch_status 178] [45048] [{'status': '1'}]
[2022-01-01 16:49:16,485 INFO Thread-3 test_suite_controller.py switch_status 188] [45048] [{'id': '1', 'status': '1'}]
[2022-01-01 16:49:16,486 INFO Thread-3 _internal.py _log 225] [45048] [127.0.0.1 - - [01/Jan/2022 16:49:16] "PATCH /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:49:18,428 INFO Thread-4 test_suite_controller.py patch 319] [39400] [传入的禁用启用的测试套件为：{'id': '1', 'project_id': 1}]
[2022-01-01 16:49:18,442 INFO Thread-4 test_suite_controller.py switch_status 164] [39400] [<TestSuiteModel> '登陆2']
[2022-01-01 16:49:18,442 INFO Thread-4 test_suite_controller.py switch_status 178] [39400] [{'status': '0'}]
[2022-01-01 16:49:18,473 INFO Thread-4 test_suite_controller.py switch_status 188] [39400] [{'id': '1', 'status': '0'}]
[2022-01-01 16:49:18,474 INFO Thread-4 _internal.py _log 225] [39400] [127.0.0.1 - - [01/Jan/2022 16:49:18] "PATCH /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:49:23,646 INFO Thread-5 test_suite_controller.py patch 319] [50928] [传入的禁用启用的测试套件为：{'id': '1', 'project_id': 1}]
[2022-01-01 16:49:23,657 INFO Thread-5 test_suite_controller.py switch_status 164] [50928] [<TestSuiteModel> '登陆2']
[2022-01-01 16:49:23,658 INFO Thread-5 test_suite_controller.py switch_status 178] [50928] [{'status': '1'}]
[2022-01-01 16:49:23,677 INFO Thread-5 test_suite_controller.py switch_status 188] [50928] [{'id': '1', 'status': '1'}]
[2022-01-01 16:49:23,678 INFO Thread-5 _internal.py _log 225] [50928] [127.0.0.1 - - [01/Jan/2022 16:49:23] "PATCH /api/suite HTTP/1.1" 200 -]
[2022-01-01 16:52:20,495 INFO Thread-6 __init__.py abort_404 53] [49684] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 16:52:20,495 INFO Thread-6 _internal.py _log 225] [49684] [127.0.0.1 - - [01/Jan/2022 16:52:20] "GET /api/testsuite?project_id=1&page=1&size=10&type=query_list HTTP/1.1" 200 -]
[2022-01-01 16:52:20,649 INFO Thread-7 __init__.py abort_404 53] [43740] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 16:52:20,650 INFO Thread-7 _internal.py _log 225] [43740] [127.0.0.1 - - [01/Jan/2022 16:52:20] "GET /favicon.ico HTTP/1.1" 200 -]
[2022-01-01 16:57:40,843 INFO MainThread _internal.py _log 225] [50044] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\execute_testcase_controller.py', reloading]
[2022-01-01 16:57:40,951 INFO MainThread _internal.py _log 225] [26664] [ * Restarting with stat]
[2022-01-01 16:57:43,788 INFO MainThread cron_utils.py start_cron 213] [47896] [开始定时任务]
[2022-01-01 16:57:43,875 INFO MainThread base.py start 166] [47896] [Scheduler started]
[2022-01-01 16:57:43,905 WARNING MainThread _internal.py _log 225] [47896] [ * Debugger is active!]
[2022-01-01 16:57:43,933 INFO MainThread _internal.py _log 225] [47896] [ * Debugger PIN: 133-332-853]
[2022-01-01 16:57:44,210 INFO Thread-1 _internal.py _log 225] [44112] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:04:17,393 INFO MainThread _internal.py _log 225] [47896] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_env_controller.py', reloading]
[2022-01-01 17:04:17,502 INFO MainThread _internal.py _log 225] [26664] [ * Restarting with stat]
[2022-01-01 17:04:21,390 INFO MainThread cron_utils.py start_cron 213] [36904] [开始定时任务]
[2022-01-01 17:04:21,506 INFO MainThread base.py start 166] [36904] [Scheduler started]
[2022-01-01 17:04:21,546 WARNING MainThread _internal.py _log 225] [36904] [ * Debugger is active!]
[2022-01-01 17:04:21,579 INFO MainThread _internal.py _log 225] [36904] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:04:21,972 INFO Thread-1 _internal.py _log 225] [26896] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:37:30,856 INFO MainThread cron_utils.py start_cron 213] [48564] [开始定时任务]
[2022-01-01 17:37:30,941 INFO MainThread base.py start 166] [48564] [Scheduler started]
[2022-01-01 17:37:30,990 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:37:35,054 INFO MainThread cron_utils.py start_cron 213] [33480] [开始定时任务]
[2022-01-01 17:37:35,133 INFO MainThread base.py start 166] [33480] [Scheduler started]
[2022-01-01 17:37:35,174 WARNING MainThread _internal.py _log 225] [33480] [ * Debugger is active!]
[2022-01-01 17:37:35,212 INFO MainThread _internal.py _log 225] [33480] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:37:35,610 INFO Thread-1 _internal.py _log 225] [50052] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:43:39,301 INFO MainThread _internal.py _log 225] [33480] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:43:39,494 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:43:44,059 INFO MainThread cron_utils.py start_cron 213] [46496] [开始定时任务]
[2022-01-01 17:43:44,176 INFO MainThread base.py start 166] [46496] [Scheduler started]
[2022-01-01 17:43:44,226 WARNING MainThread _internal.py _log 225] [46496] [ * Debugger is active!]
[2022-01-01 17:43:44,278 INFO MainThread _internal.py _log 225] [46496] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:43:44,806 INFO Thread-1 _internal.py _log 225] [30124] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:43:48,042 INFO MainThread _internal.py _log 225] [46496] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:43:48,224 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:43:53,122 INFO MainThread cron_utils.py start_cron 213] [24416] [开始定时任务]
[2022-01-01 17:43:53,221 INFO MainThread base.py start 166] [24416] [Scheduler started]
[2022-01-01 17:43:53,266 WARNING MainThread _internal.py _log 225] [24416] [ * Debugger is active!]
[2022-01-01 17:43:53,308 INFO MainThread _internal.py _log 225] [24416] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:43:53,714 INFO Thread-1 _internal.py _log 225] [46260] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:44:16,094 INFO MainThread _internal.py _log 225] [24416] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:44:16,280 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:44:21,543 INFO MainThread cron_utils.py start_cron 213] [46284] [开始定时任务]
[2022-01-01 17:44:21,639 INFO MainThread base.py start 166] [46284] [Scheduler started]
[2022-01-01 17:44:21,690 WARNING MainThread _internal.py _log 225] [46284] [ * Debugger is active!]
[2022-01-01 17:44:21,736 INFO MainThread _internal.py _log 225] [46284] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:44:22,208 INFO Thread-1 _internal.py _log 225] [44592] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:44:58,395 INFO MainThread _internal.py _log 225] [46284] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:44:58,575 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:45:03,674 INFO MainThread cron_utils.py start_cron 213] [27856] [开始定时任务]
[2022-01-01 17:45:03,792 INFO MainThread base.py start 166] [27856] [Scheduler started]
[2022-01-01 17:45:03,848 WARNING MainThread _internal.py _log 225] [27856] [ * Debugger is active!]
[2022-01-01 17:45:03,902 INFO MainThread _internal.py _log 225] [27856] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:45:04,350 INFO Thread-1 _internal.py _log 225] [608] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:45:38,822 INFO MainThread _internal.py _log 225] [27856] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:45:38,997 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:45:43,633 INFO MainThread cron_utils.py start_cron 213] [48176] [开始定时任务]
[2022-01-01 17:45:43,734 INFO MainThread base.py start 166] [48176] [Scheduler started]
[2022-01-01 17:45:43,782 WARNING MainThread _internal.py _log 225] [48176] [ * Debugger is active!]
[2022-01-01 17:45:43,826 INFO MainThread _internal.py _log 225] [48176] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:45:44,345 INFO Thread-1 _internal.py _log 225] [50364] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:45:50,609 INFO MainThread _internal.py _log 225] [48176] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 17:45:50,786 INFO MainThread _internal.py _log 225] [48564] [ * Restarting with stat]
[2022-01-01 17:45:55,932 INFO MainThread cron_utils.py start_cron 213] [45176] [开始定时任务]
[2022-01-01 17:45:56,020 INFO MainThread base.py start 166] [45176] [Scheduler started]
[2022-01-01 17:45:56,081 WARNING MainThread _internal.py _log 225] [45176] [ * Debugger is active!]
[2022-01-01 17:45:56,140 INFO MainThread _internal.py _log 225] [45176] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:45:56,716 INFO Thread-1 _internal.py _log 225] [45804] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:48:21,951 INFO MainThread cron_utils.py start_cron 213] [48028] [开始定时任务]
[2022-01-01 17:48:22,036 INFO MainThread base.py start 166] [48028] [Scheduler started]
[2022-01-01 17:48:22,083 INFO MainThread _internal.py _log 225] [48028] [ * Restarting with stat]
[2022-01-01 17:48:25,842 INFO MainThread cron_utils.py start_cron 213] [47200] [开始定时任务]
[2022-01-01 17:48:25,936 INFO MainThread base.py start 166] [47200] [Scheduler started]
[2022-01-01 17:48:25,981 WARNING MainThread _internal.py _log 225] [47200] [ * Debugger is active!]
[2022-01-01 17:48:26,018 INFO MainThread _internal.py _log 225] [47200] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:48:26,380 INFO Thread-1 _internal.py _log 225] [47468] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:48:29,235 INFO Thread-2 __init__.py abort_404 53] [31748] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 17:48:29,236 INFO Thread-2 _internal.py _log 225] [31748] [127.0.0.1 - - [01/Jan/2022 17:48:29] "POST /api/user HTTP/1.1" 200 -]
[2022-01-01 17:49:04,984 INFO MainThread _internal.py _log 225] [47200] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 17:49:05,157 INFO MainThread _internal.py _log 225] [48028] [ * Restarting with stat]
[2022-01-01 17:49:10,021 INFO MainThread cron_utils.py start_cron 213] [48152] [开始定时任务]
[2022-01-01 17:49:10,113 INFO MainThread base.py start 166] [48152] [Scheduler started]
[2022-01-01 17:49:10,158 WARNING MainThread _internal.py _log 225] [48152] [ * Debugger is active!]
[2022-01-01 17:49:10,202 INFO MainThread _internal.py _log 225] [48152] [ * Debugger PIN: 133-332-853]
[2022-01-01 17:49:10,639 INFO Thread-1 _internal.py _log 225] [36032] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 17:49:20,520 INFO Thread-2 _internal.py _log 225] [37536] [127.0.0.1 - - [01/Jan/2022 17:49:20] "POST /api/user HTTP/1.1" 200 -]
[2022-01-01 19:20:24,711 INFO MainThread _internal.py _log 225] [48152] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:20:24,922 INFO MainThread _internal.py _log 225] [48028] [ * Restarting with stat]
[2022-01-01 19:36:05,022 INFO MainThread cron_utils.py start_cron 213] [42660] [开始定时任务]
[2022-01-01 19:36:06,620 INFO MainThread base.py start 166] [42660] [Scheduler started]
[2022-01-01 19:36:06,659 INFO MainThread _internal.py _log 225] [42660] [ * Restarting with stat]
[2022-01-01 19:36:12,138 INFO MainThread cron_utils.py start_cron 213] [13192] [开始定时任务]
[2022-01-01 19:36:13,775 INFO MainThread base.py start 166] [13192] [Scheduler started]
[2022-01-01 19:36:13,806 WARNING MainThread _internal.py _log 225] [13192] [ * Debugger is active!]
[2022-01-01 19:36:13,838 INFO MainThread _internal.py _log 225] [13192] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:36:14,160 INFO Thread-1 _internal.py _log 225] [49368] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:36:43,245 INFO MainThread _internal.py _log 225] [13192] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 19:36:43,358 INFO MainThread _internal.py _log 225] [42660] [ * Restarting with stat]
[2022-01-01 19:36:49,890 INFO MainThread cron_utils.py start_cron 213] [28036] [开始定时任务]
[2022-01-01 19:36:52,423 INFO MainThread base.py start 166] [28036] [Scheduler started]
[2022-01-01 19:36:52,467 WARNING MainThread _internal.py _log 225] [28036] [ * Debugger is active!]
[2022-01-01 19:36:52,500 INFO MainThread _internal.py _log 225] [28036] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:36:52,822 INFO Thread-1 _internal.py _log 225] [49176] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:37:43,800 INFO Thread-2 user_controller.py post 73] [40404] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:37:43,840 INFO Thread-2 user_controller.py query_user 28] [40404] [当前登陆的用户为：None]
[2022-01-01 19:37:43,840 INFO Thread-2 __init__.py framework_error 73] [40404] ['NoneType' object has no attribute 'to_idct']
[2022-01-01 19:37:44,304 INFO Thread-2 _internal.py _log 225] [40404] [127.0.0.1 - - [01/Jan/2022 19:37:44] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:38:01,711 INFO MainThread _internal.py _log 225] [28036] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:38:01,820 INFO MainThread _internal.py _log 225] [42660] [ * Restarting with stat]
[2022-01-01 19:38:09,931 INFO MainThread cron_utils.py start_cron 213] [18708] [开始定时任务]
[2022-01-01 19:38:12,711 INFO MainThread base.py start 166] [18708] [Scheduler started]
[2022-01-01 19:38:12,741 WARNING MainThread _internal.py _log 225] [18708] [ * Debugger is active!]
[2022-01-01 19:38:12,772 INFO MainThread _internal.py _log 225] [18708] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:38:13,127 INFO Thread-1 _internal.py _log 225] [46336] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:38:14,472 INFO Thread-2 user_controller.py post 73] [49492] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:38:14,580 INFO Thread-2 user_controller.py query_user 28] [49492] [当前登陆的用户为：None]
[2022-01-01 19:38:14,580 INFO Thread-2 __init__.py framework_error 73] [49492] ['NoneType' object has no attribute 'to_dict']
[2022-01-01 19:38:15,026 INFO Thread-2 _internal.py _log 225] [49492] [127.0.0.1 - - [01/Jan/2022 19:38:15] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:38:29,031 INFO Thread-3 user_controller.py post 73] [50336] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:38:29,164 INFO Thread-3 user_controller.py query_user 28] [50336] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:38:29,164 INFO Thread-3 __init__.py framework_error 73] [50336] ['Flask' object has no attribute 'login_manager']
[2022-01-01 19:38:29,630 INFO Thread-3 _internal.py _log 225] [50336] [127.0.0.1 - - [01/Jan/2022 19:38:29] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:39:29,112 INFO MainThread _internal.py _log 225] [18708] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:39:29,231 INFO MainThread _internal.py _log 225] [42660] [ * Restarting with stat]
[2022-01-01 19:39:36,907 INFO MainThread cron_utils.py start_cron 213] [26616] [开始定时任务]
[2022-01-01 19:39:41,571 INFO MainThread base.py start 166] [26616] [Scheduler started]
[2022-01-01 19:39:41,611 INFO MainThread _internal.py _log 225] [26616] [ * Restarting with stat]
[2022-01-01 19:39:49,414 INFO MainThread cron_utils.py start_cron 213] [44396] [开始定时任务]
[2022-01-01 19:39:52,716 INFO MainThread base.py start 166] [44396] [Scheduler started]
[2022-01-01 19:39:52,746 WARNING MainThread _internal.py _log 225] [44396] [ * Debugger is active!]
[2022-01-01 19:39:52,774 INFO MainThread _internal.py _log 225] [44396] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:39:53,063 INFO Thread-1 _internal.py _log 225] [48380] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:39:56,114 INFO Thread-2 user_controller.py post 73] [44856] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:39:56,258 INFO Thread-2 user_controller.py query_user 28] [44856] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:39:56,258 INFO Thread-2 __init__.py framework_error 77] [44856] [The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.]
[2022-01-01 19:39:56,501 INFO Thread-2 _internal.py _log 225] [44856] [127.0.0.1 - - [01/Jan/2022 19:39:56] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:40:16,814 INFO MainThread _internal.py _log 225] [44396] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:40:16,917 INFO MainThread _internal.py _log 225] [26616] [ * Restarting with stat]
[2022-01-01 19:40:41,123 INFO MainThread cron_utils.py start_cron 213] [49780] [开始定时任务]
[2022-01-01 19:40:43,487 INFO MainThread base.py start 166] [49780] [Scheduler started]
[2022-01-01 19:40:43,519 INFO MainThread _internal.py _log 225] [49780] [ * Restarting with stat]
[2022-01-01 19:40:50,322 INFO MainThread cron_utils.py start_cron 213] [44616] [开始定时任务]
[2022-01-01 19:40:51,991 INFO MainThread base.py start 166] [44616] [Scheduler started]
[2022-01-01 19:40:52,030 WARNING MainThread _internal.py _log 225] [44616] [ * Debugger is active!]
[2022-01-01 19:40:52,066 INFO MainThread _internal.py _log 225] [44616] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:40:52,393 INFO Thread-1 _internal.py _log 225] [36292] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:40:53,035 INFO Thread-2 user_controller.py post 73] [47708] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:40:53,179 INFO Thread-2 user_controller.py query_user 28] [47708] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:40:53,653 INFO Thread-2 _internal.py _log 225] [47708] [127.0.0.1 - - [01/Jan/2022 19:40:53] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:41:02,004 INFO Thread-3 user_controller.py post 73] [46172] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:41:02,043 INFO Thread-3 user_controller.py query_user 28] [46172] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:41:02,733 INFO Thread-3 _internal.py _log 225] [46172] [127.0.0.1 - - [01/Jan/2022 19:41:02] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:41:28,147 INFO MainThread _internal.py _log 225] [44616] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:41:28,266 INFO MainThread _internal.py _log 225] [49780] [ * Restarting with stat]
[2022-01-01 19:41:36,166 INFO MainThread cron_utils.py start_cron 213] [50400] [开始定时任务]
[2022-01-01 19:41:38,477 INFO MainThread base.py start 166] [50400] [Scheduler started]
[2022-01-01 19:41:38,508 WARNING MainThread _internal.py _log 225] [50400] [ * Debugger is active!]
[2022-01-01 19:41:38,542 INFO MainThread _internal.py _log 225] [50400] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:41:38,856 INFO Thread-1 _internal.py _log 225] [10396] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:43:33,300 INFO MainThread cron_utils.py start_cron 213] [49412] [开始定时任务]
[2022-01-01 19:43:36,287 INFO MainThread base.py start 166] [49412] [Scheduler started]
[2022-01-01 19:43:36,327 INFO MainThread _internal.py _log 225] [49412] [ * Restarting with stat]
[2022-01-01 19:43:43,817 INFO MainThread cron_utils.py start_cron 213] [31904] [开始定时任务]
[2022-01-01 19:43:46,095 INFO MainThread base.py start 166] [31904] [Scheduler started]
[2022-01-01 19:43:46,124 WARNING MainThread _internal.py _log 225] [31904] [ * Debugger is active!]
[2022-01-01 19:43:46,156 INFO MainThread _internal.py _log 225] [31904] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:43:46,429 INFO Thread-1 _internal.py _log 225] [14676] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:44:15,476 INFO MainThread cron_utils.py start_cron 213] [48168] [开始定时任务]
[2022-01-01 19:44:18,233 INFO MainThread base.py start 166] [48168] [Scheduler started]
[2022-01-01 19:44:18,287 INFO MainThread _internal.py _log 225] [48168] [ * Restarting with stat]
[2022-01-01 19:44:27,067 INFO MainThread cron_utils.py start_cron 213] [46740] [开始定时任务]
[2022-01-01 19:44:29,409 INFO MainThread base.py start 166] [46740] [Scheduler started]
[2022-01-01 19:44:29,458 WARNING MainThread _internal.py _log 225] [46740] [ * Debugger is active!]
[2022-01-01 19:44:29,512 INFO MainThread _internal.py _log 225] [46740] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:44:30,191 INFO Thread-1 _internal.py _log 225] [47816] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:44:38,960 INFO Thread-2 __init__.py abort_404 53] [3484] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 19:44:38,962 INFO Thread-2 _internal.py _log 225] [3484] [127.0.0.1 - - [01/Jan/2022 19:44:38] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:45:09,652 INFO MainThread _internal.py _log 225] [46740] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 19:45:09,829 INFO MainThread _internal.py _log 225] [48168] [ * Restarting with stat]
[2022-01-01 19:45:19,081 INFO MainThread cron_utils.py start_cron 213] [46488] [开始定时任务]
[2022-01-01 19:45:22,059 INFO MainThread base.py start 166] [46488] [Scheduler started]
[2022-01-01 19:45:22,108 WARNING MainThread _internal.py _log 225] [46488] [ * Debugger is active!]
[2022-01-01 19:45:22,152 INFO MainThread _internal.py _log 225] [46488] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:45:22,645 INFO Thread-1 _internal.py _log 225] [50924] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:45:28,167 INFO Thread-2 user_controller.py post 73] [47884] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:45:28,227 INFO Thread-2 user_controller.py query_user 28] [47884] [当前登陆的用户为：None]
[2022-01-01 19:45:28,227 INFO Thread-2 __init__.py framework_error 73] [47884] ['NoneType' object has no attribute 'to_idct']
[2022-01-01 19:45:28,458 INFO Thread-2 _internal.py _log 225] [47884] [127.0.0.1 - - [01/Jan/2022 19:45:28] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:45:51,459 INFO MainThread _internal.py _log 225] [46488] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:45:51,665 INFO MainThread _internal.py _log 225] [48168] [ * Restarting with stat]
[2022-01-01 19:46:01,642 INFO MainThread cron_utils.py start_cron 213] [46908] [开始定时任务]
[2022-01-01 19:46:04,162 INFO MainThread base.py start 166] [46908] [Scheduler started]
[2022-01-01 19:46:04,216 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:46:11,393 INFO MainThread cron_utils.py start_cron 213] [49740] [开始定时任务]
[2022-01-01 19:46:13,442 INFO MainThread base.py start 166] [49740] [Scheduler started]
[2022-01-01 19:46:13,493 WARNING MainThread _internal.py _log 225] [49740] [ * Debugger is active!]
[2022-01-01 19:46:13,541 INFO MainThread _internal.py _log 225] [49740] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:46:14,026 INFO Thread-1 _internal.py _log 225] [48300] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:46:17,699 INFO Thread-2 user_controller.py post 73] [10396] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:46:17,826 INFO Thread-2 user_controller.py query_user 28] [10396] [当前登陆的用户为：None]
[2022-01-01 19:46:17,827 INFO Thread-2 __init__.py framework_error 73] [10396] ['NoneType' object has no attribute 'to_dict']
[2022-01-01 19:46:18,055 INFO Thread-2 _internal.py _log 225] [10396] [127.0.0.1 - - [01/Jan/2022 19:46:18] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:47:21,131 INFO MainThread _internal.py _log 225] [49740] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:47:21,309 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:47:30,166 INFO MainThread cron_utils.py start_cron 213] [6168] [开始定时任务]
[2022-01-01 19:47:32,459 INFO MainThread base.py start 166] [6168] [Scheduler started]
[2022-01-01 19:47:32,506 WARNING MainThread _internal.py _log 225] [6168] [ * Debugger is active!]
[2022-01-01 19:47:32,550 INFO MainThread _internal.py _log 225] [6168] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:47:32,968 INFO Thread-1 _internal.py _log 225] [41392] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:47:36,496 INFO Thread-2 user_controller.py post 76] [50876] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:47:36,764 INFO Thread-2 user_controller.py query_user 28] [50876] [当前登陆的用户为：None]
[2022-01-01 19:47:36,765 INFO Thread-2 __init__.py framework_error 73] [50876] [string indices must be integers]
[2022-01-01 19:47:36,967 INFO Thread-2 _internal.py _log 225] [50876] [127.0.0.1 - - [01/Jan/2022 19:47:36] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:48:17,875 INFO MainThread _internal.py _log 225] [6168] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:48:18,051 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:48:29,104 INFO MainThread cron_utils.py start_cron 213] [27960] [开始定时任务]
[2022-01-01 19:48:32,339 INFO MainThread base.py start 166] [27960] [Scheduler started]
[2022-01-01 19:48:32,398 WARNING MainThread _internal.py _log 225] [27960] [ * Debugger is active!]
[2022-01-01 19:48:32,452 INFO MainThread _internal.py _log 225] [27960] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:48:33,093 INFO Thread-1 _internal.py _log 225] [39320] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:48:33,431 INFO MainThread _internal.py _log 225] [27960] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:48:33,681 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:48:43,976 INFO MainThread cron_utils.py start_cron 213] [50004] [开始定时任务]
[2022-01-01 19:48:46,658 INFO MainThread base.py start 166] [50004] [Scheduler started]
[2022-01-01 19:48:46,703 WARNING MainThread _internal.py _log 225] [50004] [ * Debugger is active!]
[2022-01-01 19:48:46,747 INFO MainThread _internal.py _log 225] [50004] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:48:47,151 INFO Thread-1 _internal.py _log 225] [47704] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:48:49,976 INFO Thread-2 user_controller.py post 76] [47816] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:48:50,139 INFO Thread-2 user_controller.py query_user 28] [47816] [当前登陆的用户为：None]
[2022-01-01 19:48:50,605 INFO Thread-2 _internal.py _log 225] [47816] [127.0.0.1 - - [01/Jan/2022 19:48:50] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:49:03,007 INFO Thread-3 user_controller.py post 76] [38712] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:49:03,063 INFO Thread-3 user_controller.py query_user 28] [38712] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:49:03,063 INFO Thread-3 __init__.py framework_error 73] [38712] ['Flask' object has no attribute 'login_manager']
[2022-01-01 19:49:03,520 INFO Thread-3 _internal.py _log 225] [38712] [127.0.0.1 - - [01/Jan/2022 19:49:03] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:50:38,158 INFO MainThread _internal.py _log 225] [50004] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:50:38,267 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:50:45,820 INFO MainThread cron_utils.py start_cron 213] [50544] [开始定时任务]
[2022-01-01 19:50:48,103 INFO MainThread base.py start 166] [50544] [Scheduler started]
[2022-01-01 19:50:48,138 WARNING MainThread _internal.py _log 225] [50544] [ * Debugger is active!]
[2022-01-01 19:50:48,173 INFO MainThread _internal.py _log 225] [50544] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:50:48,506 INFO Thread-1 _internal.py _log 225] [30124] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:51:12,049 INFO MainThread _internal.py _log 225] [50544] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:51:12,153 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:51:19,353 INFO MainThread cron_utils.py start_cron 213] [45104] [开始定时任务]
[2022-01-01 19:51:23,346 INFO MainThread base.py start 166] [45104] [Scheduler started]
[2022-01-01 19:51:23,384 WARNING MainThread _internal.py _log 225] [45104] [ * Debugger is active!]
[2022-01-01 19:51:23,422 INFO MainThread _internal.py _log 225] [45104] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:51:23,813 INFO Thread-1 _internal.py _log 225] [50692] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:51:30,710 INFO MainThread _internal.py _log 225] [45104] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 19:51:30,849 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:51:37,579 INFO MainThread cron_utils.py start_cron 213] [46964] [开始定时任务]
[2022-01-01 19:51:39,665 INFO MainThread base.py start 166] [46964] [Scheduler started]
[2022-01-01 19:51:39,697 WARNING MainThread _internal.py _log 225] [46964] [ * Debugger is active!]
[2022-01-01 19:51:39,728 INFO MainThread _internal.py _log 225] [46964] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:51:40,066 INFO Thread-1 _internal.py _log 225] [34760] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:52:02,724 INFO MainThread _internal.py _log 225] [46964] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:52:02,841 INFO MainThread _internal.py _log 225] [46908] [ * Restarting with stat]
[2022-01-01 19:52:11,003 INFO MainThread cron_utils.py start_cron 213] [48376] [开始定时任务]
[2022-01-01 19:52:19,611 INFO MainThread cron_utils.py start_cron 213] [45684] [开始定时任务]
[2022-01-01 19:52:22,829 INFO MainThread base.py start 166] [45684] [Scheduler started]
[2022-01-01 19:52:22,867 INFO MainThread _internal.py _log 225] [45684] [ * Restarting with stat]
[2022-01-01 19:52:29,698 INFO MainThread cron_utils.py start_cron 213] [48380] [开始定时任务]
[2022-01-01 19:52:34,130 INFO MainThread base.py start 166] [48380] [Scheduler started]
[2022-01-01 19:52:34,191 WARNING MainThread _internal.py _log 225] [48380] [ * Debugger is active!]
[2022-01-01 19:52:34,249 INFO MainThread _internal.py _log 225] [48380] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:52:34,601 INFO Thread-1 _internal.py _log 225] [49952] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:52:36,827 INFO Thread-2 __init__.py abort_404 53] [10396] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 19:52:36,828 INFO Thread-2 _internal.py _log 225] [10396] [127.0.0.1 - - [01/Jan/2022 19:52:36] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:52:47,893 INFO MainThread _internal.py _log 225] [48380] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:52:48,034 INFO MainThread _internal.py _log 225] [45684] [ * Restarting with stat]
[2022-01-01 19:52:54,374 INFO MainThread cron_utils.py start_cron 213] [41484] [开始定时任务]
[2022-01-01 19:52:55,994 INFO MainThread base.py start 166] [41484] [Scheduler started]
[2022-01-01 19:52:56,022 WARNING MainThread _internal.py _log 225] [41484] [ * Debugger is active!]
[2022-01-01 19:52:56,051 INFO MainThread _internal.py _log 225] [41484] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:52:56,331 INFO Thread-1 _internal.py _log 225] [47948] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:53:09,469 INFO MainThread _internal.py _log 225] [41484] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:53:09,585 INFO MainThread _internal.py _log 225] [45684] [ * Restarting with stat]
[2022-01-01 19:53:17,882 INFO MainThread cron_utils.py start_cron 213] [51040] [开始定时任务]
[2022-01-01 19:53:17,998 INFO MainThread base.py start 166] [51040] [Scheduler started]
[2022-01-01 19:53:18,045 WARNING MainThread _internal.py _log 225] [51040] [ * Debugger is active!]
[2022-01-01 19:53:18,088 INFO MainThread _internal.py _log 225] [51040] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:53:18,600 INFO Thread-1 _internal.py _log 225] [23208] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:53:33,446 INFO MainThread cron_utils.py start_cron 213] [45104] [开始定时任务]
[2022-01-01 19:53:35,588 INFO MainThread base.py start 166] [45104] [Scheduler started]
[2022-01-01 19:53:35,635 INFO MainThread _internal.py _log 225] [45104] [ * Restarting with stat]
[2022-01-01 19:53:44,530 INFO MainThread cron_utils.py start_cron 213] [50220] [开始定时任务]
[2022-01-01 19:53:47,754 INFO MainThread base.py start 166] [50220] [Scheduler started]
[2022-01-01 19:53:47,794 WARNING MainThread _internal.py _log 225] [50220] [ * Debugger is active!]
[2022-01-01 19:53:47,833 INFO MainThread _internal.py _log 225] [50220] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:53:48,257 INFO Thread-1 _internal.py _log 225] [42396] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:54:15,333 INFO Thread-2 __init__.py abort_404 53] [44428] [404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.]
[2022-01-01 19:54:15,334 INFO Thread-2 _internal.py _log 225] [44428] [127.0.0.1 - - [01/Jan/2022 19:54:15] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:54:45,178 INFO MainThread _internal.py _log 225] [50220] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 19:54:45,365 INFO MainThread _internal.py _log 225] [45104] [ * Restarting with stat]
[2022-01-01 19:54:53,863 INFO MainThread cron_utils.py start_cron 213] [43684] [开始定时任务]
[2022-01-01 19:54:57,058 INFO MainThread base.py start 166] [43684] [Scheduler started]
[2022-01-01 19:55:59,027 INFO MainThread cron_utils.py start_cron 213] [46536] [开始定时任务]
[2022-01-01 19:56:01,208 INFO MainThread base.py start 166] [46536] [Scheduler started]
[2022-01-01 19:56:01,268 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 19:56:10,021 INFO MainThread cron_utils.py start_cron 213] [44264] [开始定时任务]
[2022-01-01 19:56:12,998 INFO MainThread base.py start 166] [44264] [Scheduler started]
[2022-01-01 19:56:13,046 WARNING MainThread _internal.py _log 225] [44264] [ * Debugger is active!]
[2022-01-01 19:56:13,089 INFO MainThread _internal.py _log 225] [44264] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:56:13,502 INFO Thread-1 _internal.py _log 225] [43868] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:56:20,621 INFO Thread-2 user_controller.py post 73] [46656] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138007",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:56:20,865 INFO Thread-2 user_controller.py query_user 28] [46656] [当前登陆的用户为：None]
[2022-01-01 19:56:20,865 INFO Thread-2 __init__.py framework_error 73] [46656] ['NoneType' object has no attribute 'to_idct']
[2022-01-01 19:56:21,313 INFO Thread-2 _internal.py _log 225] [46656] [127.0.0.1 - - [01/Jan/2022 19:56:21] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:56:54,000 INFO MainThread _internal.py _log 225] [44264] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 19:56:54,177 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 19:57:03,665 INFO MainThread cron_utils.py start_cron 213] [36292] [开始定时任务]
[2022-01-01 19:57:06,204 INFO MainThread base.py start 166] [36292] [Scheduler started]
[2022-01-01 19:57:06,261 WARNING MainThread _internal.py _log 225] [36292] [ * Debugger is active!]
[2022-01-01 19:57:06,312 INFO MainThread _internal.py _log 225] [36292] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:57:06,781 INFO Thread-1 _internal.py _log 225] [48860] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:57:21,249 INFO Thread-2 user_controller.py post 73] [36984] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"1380000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:57:21,485 INFO Thread-2 user_controller.py query_user 28] [36984] [当前登陆的用户为：None]
[2022-01-01 19:57:21,485 INFO Thread-2 __init__.py framework_error 73] [36984] ['NoneType' object has no attribute 'to_dict']
[2022-01-01 19:57:21,941 INFO Thread-2 _internal.py _log 225] [36984] [127.0.0.1 - - [01/Jan/2022 19:57:21] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:57:40,607 INFO Thread-3 user_controller.py post 73] [49132] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:57:40,713 INFO Thread-3 user_controller.py query_user 28] [49132] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:57:40,714 INFO Thread-3 __init__.py framework_error 73] [49132] ['Flask' object has no attribute 'login_manager']
[2022-01-01 19:57:41,197 INFO Thread-3 _internal.py _log 225] [49132] [127.0.0.1 - - [01/Jan/2022 19:57:41] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 19:59:38,554 INFO MainThread _internal.py _log 225] [36292] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 19:59:38,722 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 19:59:47,445 INFO MainThread cron_utils.py start_cron 213] [37804] [开始定时任务]
[2022-01-01 19:59:49,762 INFO MainThread base.py start 166] [37804] [Scheduler started]
[2022-01-01 19:59:49,811 WARNING MainThread _internal.py _log 225] [37804] [ * Debugger is active!]
[2022-01-01 19:59:49,862 INFO MainThread _internal.py _log 225] [37804] [ * Debugger PIN: 133-332-853]
[2022-01-01 19:59:50,465 INFO Thread-1 _internal.py _log 225] [41564] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 19:59:56,897 INFO Thread-2 user_controller.py post 73] [39480] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 19:59:57,064 INFO Thread-2 user_controller.py query_user 28] [39480] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 19:59:57,065 INFO Thread-2 __init__.py framework_error 75] [39480] [The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.]
[2022-01-01 19:59:57,300 INFO Thread-2 _internal.py _log 225] [39480] [127.0.0.1 - - [01/Jan/2022 19:59:57] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:00:42,938 INFO MainThread _internal.py _log 225] [37804] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 20:00:43,122 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:00:52,571 INFO MainThread cron_utils.py start_cron 213] [50640] [开始定时任务]
[2022-01-01 20:00:56,728 INFO MainThread base.py start 166] [50640] [Scheduler started]
[2022-01-01 20:00:56,771 WARNING MainThread _internal.py _log 225] [50640] [ * Debugger is active!]
[2022-01-01 20:00:56,811 INFO MainThread _internal.py _log 225] [50640] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:00:57,252 INFO Thread-1 _internal.py _log 225] [19268] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:01:00,811 INFO Thread-2 user_controller.py post 73] [41548] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:01:00,866 INFO Thread-2 user_controller.py query_user 28] [41548] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:01:00,867 INFO Thread-2 __init__.py framework_error 77] [41548] [The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.]
[2022-01-01 20:01:01,146 INFO Thread-2 _internal.py _log 225] [41548] [127.0.0.1 - - [01/Jan/2022 20:01:01] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:01:32,545 INFO MainThread _internal.py _log 225] [50640] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 20:01:32,749 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:01:42,032 INFO MainThread cron_utils.py start_cron 213] [31768] [开始定时任务]
[2022-01-01 20:01:44,148 INFO MainThread base.py start 166] [31768] [Scheduler started]
[2022-01-01 20:01:44,195 WARNING MainThread _internal.py _log 225] [31768] [ * Debugger is active!]
[2022-01-01 20:01:44,239 INFO MainThread _internal.py _log 225] [31768] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:01:44,754 INFO Thread-1 _internal.py _log 225] [48988] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:01:56,183 INFO Thread-2 user_controller.py post 73] [49808] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:01:56,396 INFO Thread-2 user_controller.py query_user 28] [49808] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:01:57,091 INFO Thread-2 _internal.py _log 225] [49808] [127.0.0.1 - - [01/Jan/2022 20:01:57] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:02:33,113 INFO MainThread _internal.py _log 225] [31768] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:02:33,300 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:02:43,573 INFO MainThread cron_utils.py start_cron 213] [42864] [开始定时任务]
[2022-01-01 20:02:46,367 INFO MainThread base.py start 166] [42864] [Scheduler started]
[2022-01-01 20:02:46,413 WARNING MainThread _internal.py _log 225] [42864] [ * Debugger is active!]
[2022-01-01 20:02:46,461 INFO MainThread _internal.py _log 225] [42864] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:02:46,915 INFO Thread-1 _internal.py _log 225] [49316] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:02:53,343 INFO Thread-2 user_controller.py post 73] [46452] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:02:53,621 INFO Thread-2 user_controller.py query_user 28] [46452] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:02:54,335 INFO Thread-2 _internal.py _log 225] [46452] [127.0.0.1 - - [01/Jan/2022 20:02:54] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:06:59,446 INFO MainThread _internal.py _log 225] [42864] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:06:59,566 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:07:07,696 INFO MainThread cron_utils.py start_cron 213] [37028] [开始定时任务]
[2022-01-01 20:07:10,755 INFO MainThread base.py start 166] [37028] [Scheduler started]
[2022-01-01 20:07:10,789 WARNING MainThread _internal.py _log 225] [37028] [ * Debugger is active!]
[2022-01-01 20:07:10,818 INFO MainThread _internal.py _log 225] [37028] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:07:11,160 INFO Thread-1 _internal.py _log 225] [46336] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:07:25,605 INFO MainThread _internal.py _log 225] [37028] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\run.py', reloading]
[2022-01-01 20:07:25,712 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:07:32,901 INFO MainThread cron_utils.py start_cron 213] [49392] [开始定时任务]
[2022-01-01 20:07:35,164 INFO MainThread base.py start 166] [49392] [Scheduler started]
[2022-01-01 20:07:35,199 WARNING MainThread _internal.py _log 225] [49392] [ * Debugger is active!]
[2022-01-01 20:07:35,232 INFO MainThread _internal.py _log 225] [49392] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:07:35,576 INFO Thread-1 _internal.py _log 225] [50856] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:07:59,680 INFO Thread-2 _internal.py _log 225] [43780] [127.0.0.1 - - [01/Jan/2022 20:07:59] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:08:19,353 INFO MainThread _internal.py _log 225] [49392] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:08:19,472 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:08:27,282 INFO MainThread cron_utils.py start_cron 213] [51168] [开始定时任务]
[2022-01-01 20:08:30,245 INFO MainThread base.py start 166] [51168] [Scheduler started]
[2022-01-01 20:08:30,275 WARNING MainThread _internal.py _log 225] [51168] [ * Debugger is active!]
[2022-01-01 20:08:30,304 INFO MainThread _internal.py _log 225] [51168] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:08:30,582 INFO Thread-1 _internal.py _log 225] [6412] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:08:33,134 INFO Thread-2 user_controller.py post 97] [23208] [No user_loader has been installed for this LoginManager. Refer tohttps://flask-login.readthedocs.io/en/latest/#how-it-works for more info.]
[2022-01-01 20:08:33,136 INFO Thread-2 _internal.py _log 225] [23208] [127.0.0.1 - - [01/Jan/2022 20:08:33] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:10:45,700 INFO MainThread _internal.py _log 225] [51168] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:10:45,814 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:10:53,650 INFO MainThread cron_utils.py start_cron 213] [50596] [开始定时任务]
[2022-01-01 20:10:56,438 INFO MainThread base.py start 166] [50596] [Scheduler started]
[2022-01-01 20:10:56,471 WARNING MainThread _internal.py _log 225] [50596] [ * Debugger is active!]
[2022-01-01 20:10:56,503 INFO MainThread _internal.py _log 225] [50596] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:10:56,835 INFO Thread-1 _internal.py _log 225] [2536] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:10:59,243 INFO Thread-2 user_controller.py query_user 36] [47492] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:10:59,244 INFO Thread-2 user_controller.py post 108] [47492] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:10:59,710 INFO Thread-2 _internal.py _log 225] [47492] [127.0.0.1 - - [01/Jan/2022 20:10:59] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:11:04,573 INFO Thread-3 user_controller.py post 81] [47108] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:11:04,610 INFO Thread-3 user_controller.py query_user 36] [47108] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:11:05,049 INFO Thread-3 _internal.py _log 225] [47108] [127.0.0.1 - - [01/Jan/2022 20:11:05] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:11:07,143 INFO Thread-4 user_controller.py query_user 36] [47924] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:11:07,143 INFO Thread-4 user_controller.py post 108] [47924] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:11:07,370 INFO Thread-4 _internal.py _log 225] [47924] [127.0.0.1 - - [01/Jan/2022 20:11:07] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:12:23,649 INFO MainThread _internal.py _log 225] [50596] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:12:23,799 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:12:29,359 INFO MainThread cron_utils.py start_cron 213] [36356] [开始定时任务]
[2022-01-01 20:12:31,833 INFO MainThread base.py start 166] [36356] [Scheduler started]
[2022-01-01 20:12:31,864 WARNING MainThread _internal.py _log 225] [36356] [ * Debugger is active!]
[2022-01-01 20:12:31,891 INFO MainThread _internal.py _log 225] [36356] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:12:32,172 INFO Thread-1 _internal.py _log 225] [47984] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:12:35,563 INFO Thread-2 __init__.py framework_error 77] [13820] ['KeyError' object has no attribute 'text']
[2022-01-01 20:12:35,565 INFO Thread-2 _internal.py _log 225] [13820] [127.0.0.1 - - [01/Jan/2022 20:12:35] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:12:51,022 INFO MainThread _internal.py _log 225] [36356] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:12:51,174 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:12:59,138 INFO MainThread cron_utils.py start_cron 213] [28484] [开始定时任务]
[2022-01-01 20:13:02,186 INFO MainThread base.py start 166] [28484] [Scheduler started]
[2022-01-01 20:13:02,222 WARNING MainThread _internal.py _log 225] [28484] [ * Debugger is active!]
[2022-01-01 20:13:02,257 INFO MainThread _internal.py _log 225] [28484] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:13:02,589 INFO Thread-1 _internal.py _log 225] [50600] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:13:08,469 INFO Thread-2 user_controller.py post 81] [49588] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:13:08,514 INFO Thread-2 user_controller.py query_user 36] [49588] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:13:08,988 INFO Thread-2 _internal.py _log 225] [49588] [127.0.0.1 - - [01/Jan/2022 20:13:08] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:13:11,531 INFO Thread-3 user_controller.py query_user 36] [47512] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:13:11,531 INFO Thread-3 user_controller.py post 108] [47512] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:13:11,770 INFO Thread-3 _internal.py _log 225] [47512] [127.0.0.1 - - [01/Jan/2022 20:13:11] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:13:41,394 INFO MainThread _internal.py _log 225] [28484] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:13:41,507 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:13:48,559 INFO MainThread cron_utils.py start_cron 213] [50856] [开始定时任务]
[2022-01-01 20:13:50,702 INFO MainThread base.py start 166] [50856] [Scheduler started]
[2022-01-01 20:13:50,734 WARNING MainThread _internal.py _log 225] [50856] [ * Debugger is active!]
[2022-01-01 20:13:50,764 INFO MainThread _internal.py _log 225] [50856] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:13:51,058 INFO Thread-1 _internal.py _log 225] [41924] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:13:53,746 INFO Thread-2 user_controller.py post 108] [38144] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:13:53,746 INFO Thread-2 user_controller.py post 109] [38144] [报错：    <class 'KeyError'>]
[2022-01-01 20:13:53,747 INFO Thread-2 _internal.py _log 225] [38144] [127.0.0.1 - - [01/Jan/2022 20:13:53] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:14:14,669 INFO MainThread _internal.py _log 225] [50856] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:14:14,782 INFO MainThread _internal.py _log 225] [46536] [ * Restarting with stat]
[2022-01-01 20:14:22,060 INFO MainThread cron_utils.py start_cron 213] [38332] [开始定时任务]
[2022-01-01 20:14:24,594 INFO MainThread base.py start 166] [38332] [Scheduler started]
[2022-01-01 20:14:24,631 WARNING MainThread _internal.py _log 225] [38332] [ * Debugger is active!]
[2022-01-01 20:14:24,670 INFO MainThread _internal.py _log 225] [38332] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:14:24,987 INFO Thread-1 _internal.py _log 225] [48860] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:14:51,065 INFO Thread-2 user_controller.py post 108] [37660] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:14:51,065 INFO Thread-2 user_controller.py post 109] [37660] [报错：    <class 'KeyError'>]
[2022-01-01 20:14:51,065 INFO Thread-2 user_controller.py post 110] [37660] [报错：    (<Response 91 bytes [200 OK]>,)]
[2022-01-01 20:14:51,066 INFO Thread-2 _internal.py _log 225] [37660] [127.0.0.1 - - [01/Jan/2022 20:14:51] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:18:44,425 INFO MainThread cron_utils.py start_cron 213] [36352] [开始定时任务]
[2022-01-01 20:18:44,510 INFO MainThread base.py start 166] [36352] [Scheduler started]
[2022-01-01 20:18:44,548 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:18:47,330 INFO MainThread cron_utils.py start_cron 213] [41924] [开始定时任务]
[2022-01-01 20:18:47,433 INFO MainThread base.py start 166] [41924] [Scheduler started]
[2022-01-01 20:18:47,462 WARNING MainThread _internal.py _log 225] [41924] [ * Debugger is active!]
[2022-01-01 20:18:47,489 INFO MainThread _internal.py _log 225] [41924] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:18:47,754 INFO Thread-1 _internal.py _log 225] [51188] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:18:54,034 INFO Thread-2 user_controller.py post 81] [48376] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:18:54,068 INFO Thread-2 user_controller.py query_user 28] [48376] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:18:54,089 INFO Thread-2 _internal.py _log 225] [48376] [127.0.0.1 - - [01/Jan/2022 20:18:54] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:18:56,161 INFO Thread-3 user_controller.py query_user 28] [43492] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:18:56,161 INFO Thread-3 user_controller.py post 108] [43492] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:18:56,161 INFO Thread-3 user_controller.py post 109] [43492] [报错：    <class 'KeyError'>]
[2022-01-01 20:18:56,161 INFO Thread-3 user_controller.py post 110] [43492] [报错：    (<Response 91 bytes [200 OK]>,)]
[2022-01-01 20:18:56,172 INFO Thread-3 _internal.py _log 225] [43492] [127.0.0.1 - - [01/Jan/2022 20:18:56] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:19:43,446 INFO MainThread _internal.py _log 225] [41924] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:19:43,544 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:19:46,783 INFO MainThread cron_utils.py start_cron 213] [49756] [开始定时任务]
[2022-01-01 20:19:46,897 INFO MainThread base.py start 166] [49756] [Scheduler started]
[2022-01-01 20:19:46,931 WARNING MainThread _internal.py _log 225] [49756] [ * Debugger is active!]
[2022-01-01 20:19:46,960 INFO MainThread _internal.py _log 225] [49756] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:19:47,236 INFO Thread-1 _internal.py _log 225] [46824] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:19:58,916 INFO MainThread _internal.py _log 225] [49756] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:19:59,018 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:20:02,015 INFO MainThread cron_utils.py start_cron 213] [49456] [开始定时任务]
[2022-01-01 20:20:02,113 INFO MainThread base.py start 166] [49456] [Scheduler started]
[2022-01-01 20:20:02,152 WARNING MainThread _internal.py _log 225] [49456] [ * Debugger is active!]
[2022-01-01 20:20:02,187 INFO MainThread _internal.py _log 225] [49456] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:20:02,507 INFO Thread-1 _internal.py _log 225] [608] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:20:14,260 INFO Thread-2 user_controller.py post 81] [46120] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:20:14,288 INFO Thread-2 user_controller.py query_user 28] [46120] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:20:14,307 INFO Thread-2 _internal.py _log 225] [46120] [127.0.0.1 - - [01/Jan/2022 20:20:14] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:20:16,309 INFO Thread-3 user_controller.py query_user 28] [48928] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:20:16,310 INFO Thread-3 user_controller.py post 108] [48928] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:20:16,310 INFO Thread-3 user_controller.py post 109] [48928] [报错：    <class 'KeyError'>]
[2022-01-01 20:20:16,310 INFO Thread-3 user_controller.py post 110] [48928] [报错：    (<Response 91 bytes [200 OK]>,)]
[2022-01-01 20:20:16,322 INFO Thread-3 _internal.py _log 225] [48928] [127.0.0.1 - - [01/Jan/2022 20:20:16] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:24:32,418 INFO MainThread _internal.py _log 225] [49456] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:24:32,537 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:24:35,523 INFO MainThread cron_utils.py start_cron 213] [19944] [开始定时任务]
[2022-01-01 20:24:35,676 INFO MainThread base.py start 166] [19944] [Scheduler started]
[2022-01-01 20:24:35,788 WARNING MainThread _internal.py _log 225] [19944] [ * Debugger is active!]
[2022-01-01 20:24:35,827 INFO MainThread _internal.py _log 225] [19944] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:24:36,181 INFO Thread-1 _internal.py _log 225] [50692] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:25:01,640 INFO MainThread _internal.py _log 225] [19944] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 20:25:01,735 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:25:04,543 INFO MainThread cron_utils.py start_cron 213] [21552] [开始定时任务]
[2022-01-01 20:25:04,612 INFO MainThread base.py start 166] [21552] [Scheduler started]
[2022-01-01 20:25:04,649 WARNING MainThread _internal.py _log 225] [21552] [ * Debugger is active!]
[2022-01-01 20:25:04,679 INFO MainThread _internal.py _log 225] [21552] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:25:04,967 INFO Thread-1 _internal.py _log 225] [46168] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:25:16,654 INFO Thread-2 user_controller.py post 86] [49716] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:25:16,685 INFO Thread-2 user_controller.py query_user 28] [49716] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:25:16,707 INFO Thread-2 _internal.py _log 225] [49716] [127.0.0.1 - - [01/Jan/2022 20:25:16] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:25:39,912 INFO Thread-3 user_controller.py query_user 28] [44136] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:25:39,913 INFO Thread-3 user_controller.py post 113] [44136] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:25:39,913 INFO Thread-3 user_controller.py post 114] [44136] [报错：    <class 'KeyError'>]
[2022-01-01 20:25:39,913 INFO Thread-3 user_controller.py post 115] [44136] [报错：    (<Response 91 bytes [200 OK]>,)]
[2022-01-01 20:25:39,923 INFO Thread-3 _internal.py _log 225] [44136] [127.0.0.1 - - [01/Jan/2022 20:25:39] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:26:26,363 INFO MainThread _internal.py _log 225] [21552] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 20:26:26,469 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:26:29,313 INFO MainThread cron_utils.py start_cron 213] [34288] [开始定时任务]
[2022-01-01 20:26:29,394 INFO MainThread base.py start 166] [34288] [Scheduler started]
[2022-01-01 20:26:29,427 WARNING MainThread _internal.py _log 225] [34288] [ * Debugger is active!]
[2022-01-01 20:26:29,455 INFO MainThread _internal.py _log 225] [34288] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:26:29,733 INFO Thread-1 _internal.py _log 225] [30736] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:26:38,770 INFO Thread-2 _internal.py _log 225] [50132] [127.0.0.1 - - [01/Jan/2022 20:26:38] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:26:44,180 INFO Thread-3 user_controller.py post 86] [51116] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:26:44,209 INFO Thread-3 user_controller.py query_user 28] [51116] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:26:44,231 INFO Thread-3 _internal.py _log 225] [51116] [127.0.0.1 - - [01/Jan/2022 20:26:44] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:26:47,317 INFO Thread-4 user_controller.py query_user 28] [46672] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:26:47,328 INFO Thread-4 test_project_controller.py query_project_by_id 29] [46672] [查询的测试计划id为：1的详情数据为：<TestProjectModel> '拉勾教育测试计划']
[2022-01-01 20:26:47,328 INFO Thread-4 test_project_controller.py query_project_by_id 32] [46672] [查询的测试计划id为：1的详情数据转化为json后：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]
[2022-01-01 20:26:47,328 INFO Thread-4 test_project_controller.py get 123] [46672] [把日期对象转化为字符串之后，的结果为：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}]
[2022-01-01 20:26:47,339 INFO Thread-4 _internal.py _log 225] [46672] [127.0.0.1 - - [01/Jan/2022 20:26:47] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:26:49,865 INFO Thread-5 user_controller.py query_user 28] [47080] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:26:49,866 INFO Thread-5 user_controller.py post 113] [47080] [报错：    <Response 91 bytes [200 OK]>]
[2022-01-01 20:26:49,866 INFO Thread-5 user_controller.py post 114] [47080] [报错：    <class 'KeyError'>]
[2022-01-01 20:26:49,866 INFO Thread-5 user_controller.py post 115] [47080] [报错：    (<Response 91 bytes [200 OK]>,)]
[2022-01-01 20:26:49,878 INFO Thread-5 _internal.py _log 225] [47080] [127.0.0.1 - - [01/Jan/2022 20:26:49] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:26:52,589 INFO Thread-6 _internal.py _log 225] [50924] [127.0.0.1 - - [01/Jan/2022 20:26:52] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:27:00,744 INFO MainThread _internal.py _log 225] [34288] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 20:27:00,864 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:27:04,627 INFO MainThread cron_utils.py start_cron 213] [47984] [开始定时任务]
[2022-01-01 20:27:04,737 INFO MainThread base.py start 166] [47984] [Scheduler started]
[2022-01-01 20:27:04,779 WARNING MainThread _internal.py _log 225] [47984] [ * Debugger is active!]
[2022-01-01 20:27:04,814 INFO MainThread _internal.py _log 225] [47984] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:27:05,122 INFO Thread-1 _internal.py _log 225] [31904] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:27:47,241 INFO MainThread _internal.py _log 225] [47984] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:27:47,348 INFO MainThread _internal.py _log 225] [36352] [ * Restarting with stat]
[2022-01-01 20:27:50,317 INFO MainThread cron_utils.py start_cron 213] [30984] [开始定时任务]
[2022-01-01 20:27:50,413 INFO MainThread base.py start 166] [30984] [Scheduler started]
[2022-01-01 20:27:50,446 WARNING MainThread _internal.py _log 225] [30984] [ * Debugger is active!]
[2022-01-01 20:27:50,475 INFO MainThread _internal.py _log 225] [30984] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:27:50,750 INFO Thread-1 _internal.py _log 225] [27696] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:27:55,025 INFO Thread-2 user_controller.py post 86] [20156] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:27:55,054 INFO Thread-2 user_controller.py query_user 28] [20156] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:27:55,078 INFO Thread-2 _internal.py _log 225] [20156] [127.0.0.1 - - [01/Jan/2022 20:27:55] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:27:57,130 INFO Thread-3 user_controller.py query_user 28] [39880] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:27:57,131 INFO Thread-3 __init__.py framework_error 77] [39880] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:27:57,153 INFO Thread-3 _internal.py _log 225] [39880] [127.0.0.1 - - [01/Jan/2022 20:27:57] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:29:37,409 INFO MainThread cron_utils.py start_cron 213] [25240] [开始定时任务]
[2022-01-01 20:29:37,501 INFO MainThread base.py start 166] [25240] [Scheduler started]
[2022-01-01 20:29:37,560 INFO MainThread _internal.py _log 225] [25240] [ * Restarting with stat]
[2022-01-01 20:29:41,914 INFO MainThread cron_utils.py start_cron 213] [47064] [开始定时任务]
[2022-01-01 20:29:42,035 INFO MainThread base.py start 166] [47064] [Scheduler started]
[2022-01-01 20:29:42,089 WARNING MainThread _internal.py _log 225] [47064] [ * Debugger is active!]
[2022-01-01 20:29:42,122 INFO MainThread _internal.py _log 225] [47064] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:29:42,699 INFO Thread-6 _internal.py _log 225] [42284] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:29:55,933 INFO Thread-7 user_controller.py post 86] [44600] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:29:55,993 INFO Thread-7 user_controller.py query_user 28] [44600] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:29:56,017 INFO Thread-7 _internal.py _log 225] [44600] [127.0.0.1 - - [01/Jan/2022 20:29:56] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:30:00,495 INFO Thread-8 user_controller.py query_user 28] [34832] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:30:00,496 INFO Thread-8 __init__.py framework_error 77] [34832] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:30:00,508 INFO Thread-8 _internal.py _log 225] [34832] [127.0.0.1 - - [01/Jan/2022 20:30:00] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:31:02,388 INFO MainThread cron_utils.py start_cron 213] [48404] [开始定时任务]
[2022-01-01 20:31:02,467 INFO MainThread base.py start 166] [48404] [Scheduler started]
[2022-01-01 20:31:02,502 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:31:05,097 INFO MainThread cron_utils.py start_cron 213] [34744] [开始定时任务]
[2022-01-01 20:31:05,175 INFO MainThread base.py start 166] [34744] [Scheduler started]
[2022-01-01 20:31:05,206 WARNING MainThread _internal.py _log 225] [34744] [ * Debugger is active!]
[2022-01-01 20:31:05,234 INFO MainThread _internal.py _log 225] [34744] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:31:05,501 INFO Thread-1 _internal.py _log 225] [46308] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:31:10,426 INFO Thread-2 user_controller.py post 86] [19944] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:31:10,456 INFO Thread-2 user_controller.py query_user 28] [19944] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:31:10,476 INFO Thread-2 _internal.py _log 225] [19944] [127.0.0.1 - - [01/Jan/2022 20:31:10] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:31:12,149 INFO Thread-3 user_controller.py query_user 28] [44256] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:31:12,149 INFO Thread-3 __init__.py framework_error 77] [44256] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:31:12,161 INFO Thread-3 _internal.py _log 225] [44256] [127.0.0.1 - - [01/Jan/2022 20:31:12] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:31:42,302 INFO MainThread _internal.py _log 225] [34744] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\venv\\lib\\site-packages\\flask_login\\utils.py', reloading]
[2022-01-01 20:31:42,396 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:31:45,298 INFO MainThread cron_utils.py start_cron 213] [50060] [开始定时任务]
[2022-01-01 20:31:45,380 INFO MainThread base.py start 166] [50060] [Scheduler started]
[2022-01-01 20:31:45,416 WARNING MainThread _internal.py _log 225] [50060] [ * Debugger is active!]
[2022-01-01 20:31:45,448 INFO MainThread _internal.py _log 225] [50060] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:31:45,737 INFO Thread-1 _internal.py _log 225] [48288] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:31:48,555 INFO MainThread _internal.py _log 225] [50060] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\venv\\lib\\site-packages\\flask_login\\utils.py', reloading]
[2022-01-01 20:31:48,649 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:31:51,517 INFO MainThread cron_utils.py start_cron 213] [48928] [开始定时任务]
[2022-01-01 20:31:51,590 INFO MainThread base.py start 166] [48928] [Scheduler started]
[2022-01-01 20:31:51,621 WARNING MainThread _internal.py _log 225] [48928] [ * Debugger is active!]
[2022-01-01 20:31:51,650 INFO MainThread _internal.py _log 225] [48928] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:31:51,938 INFO Thread-1 _internal.py _log 225] [47644] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:31:54,891 INFO Thread-2 user_controller.py post 86] [39284] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:31:54,921 INFO Thread-2 user_controller.py query_user 28] [39284] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:31:54,941 INFO Thread-2 _internal.py _log 225] [39284] [127.0.0.1 - - [01/Jan/2022 20:31:54] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:31:56,594 INFO Thread-3 user_controller.py query_user 28] [45588] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:31:56,596 INFO Thread-3 __init__.py framework_error 77] [45588] [<Response 91 bytes [200 OK]>]
[2022-01-01 20:31:56,607 INFO Thread-3 _internal.py _log 225] [45588] [127.0.0.1 - - [01/Jan/2022 20:31:56] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:32:54,879 INFO MainThread _internal.py _log 225] [48928] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:32:54,991 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:32:58,130 INFO MainThread cron_utils.py start_cron 213] [50720] [开始定时任务]
[2022-01-01 20:32:58,205 INFO MainThread base.py start 166] [50720] [Scheduler started]
[2022-01-01 20:32:58,240 WARNING MainThread _internal.py _log 225] [50720] [ * Debugger is active!]
[2022-01-01 20:32:58,273 INFO MainThread _internal.py _log 225] [50720] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:32:58,564 INFO Thread-1 _internal.py _log 225] [48292] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:33:03,333 INFO Thread-2 user_controller.py post 86] [26976] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:33:03,361 INFO Thread-2 user_controller.py query_user 28] [26976] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:33:03,380 INFO Thread-2 _internal.py _log 225] [26976] [127.0.0.1 - - [01/Jan/2022 20:33:03] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:33:05,026 INFO Thread-3 user_controller.py query_user 28] [47040] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:33:05,038 INFO Thread-3 _internal.py _log 225] [47040] [127.0.0.1 - - [01/Jan/2022 20:33:05] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:33:16,729 INFO MainThread _internal.py _log 225] [50720] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\venv\\lib\\site-packages\\flask_login\\utils.py', reloading]
[2022-01-01 20:33:16,840 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:33:20,040 INFO MainThread cron_utils.py start_cron 213] [18720] [开始定时任务]
[2022-01-01 20:33:20,133 INFO MainThread base.py start 166] [18720] [Scheduler started]
[2022-01-01 20:33:20,165 WARNING MainThread _internal.py _log 225] [18720] [ * Debugger is active!]
[2022-01-01 20:33:20,196 INFO MainThread _internal.py _log 225] [18720] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:33:20,498 INFO Thread-1 _internal.py _log 225] [46840] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:34:05,499 INFO MainThread _internal.py _log 225] [18720] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\app\\__init__.py', reloading]
[2022-01-01 20:34:05,600 INFO MainThread _internal.py _log 225] [48404] [ * Restarting with stat]
[2022-01-01 20:39:47,576 INFO MainThread cron_utils.py start_cron 213] [50336] [开始定时任务]
[2022-01-01 20:39:47,698 INFO MainThread base.py start 166] [50336] [Scheduler started]
[2022-01-01 20:39:47,748 INFO MainThread _internal.py _log 225] [50336] [ * Restarting with stat]
[2022-01-01 20:39:51,493 INFO MainThread cron_utils.py start_cron 213] [46812] [开始定时任务]
[2022-01-01 20:39:51,587 INFO MainThread base.py start 166] [46812] [Scheduler started]
[2022-01-01 20:39:51,630 WARNING MainThread _internal.py _log 225] [46812] [ * Debugger is active!]
[2022-01-01 20:39:51,671 INFO MainThread _internal.py _log 225] [46812] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:39:52,052 INFO Thread-1 _internal.py _log 225] [45288] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:40:24,453 INFO Thread-2 _internal.py _log 225] [19476] [127.0.0.1 - - [01/Jan/2022 20:40:24] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:40:48,643 INFO Thread-3 _internal.py _log 225] [50132] [127.0.0.1 - - [01/Jan/2022 20:40:48] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:41:05,455 INFO Thread-4 test_project_controller.py query_project_by_id 29] [10356] [查询的测试计划id为：1的详情数据为：<TestProjectModel> '拉勾教育测试计划']
[2022-01-01 20:41:05,455 INFO Thread-4 test_project_controller.py query_project_by_id 32] [10356] [查询的测试计划id为：1的详情数据转化为json后：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]
[2022-01-01 20:41:05,456 INFO Thread-4 test_project_controller.py get 121] [10356] [把日期对象转化为字符串之后，的结果为：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}]
[2022-01-01 20:41:05,471 INFO Thread-4 _internal.py _log 225] [10356] [127.0.0.1 - - [01/Jan/2022 20:41:05] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:41:08,444 INFO Thread-5 test_project_controller.py query_project_by_id 29] [34556] [查询的测试计划id为：1的详情数据为：<TestProjectModel> '拉勾教育测试计划']
[2022-01-01 20:41:08,445 INFO Thread-5 test_project_controller.py query_project_by_id 32] [34556] [查询的测试计划id为：1的详情数据转化为json后：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]
[2022-01-01 20:41:08,445 INFO Thread-5 test_project_controller.py get 121] [34556] [把日期对象转化为字符串之后，的结果为：{'id': 1, 'project_name': '拉勾教育测试计划', 'description': '测试拉勾教育接口', 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}]
[2022-01-01 20:41:08,458 INFO Thread-5 _internal.py _log 225] [34556] [127.0.0.1 - - [01/Jan/2022 20:41:08] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:41:35,223 INFO MainThread _internal.py _log 225] [46812] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 20:41:35,379 INFO MainThread _internal.py _log 225] [50336] [ * Restarting with stat]
[2022-01-01 20:41:39,665 INFO MainThread cron_utils.py start_cron 213] [47968] [开始定时任务]
[2022-01-01 20:41:39,740 INFO MainThread base.py start 166] [47968] [Scheduler started]
[2022-01-01 20:41:39,783 WARNING MainThread _internal.py _log 225] [47968] [ * Debugger is active!]
[2022-01-01 20:41:39,822 INFO MainThread _internal.py _log 225] [47968] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:41:40,229 INFO Thread-1 _internal.py _log 225] [13700] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:41:55,025 INFO Thread-2 user_controller.py post 73] [26748] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:41:55,071 INFO Thread-2 user_controller.py query_user 28] [26748] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:41:55,096 INFO Thread-2 _internal.py _log 225] [26748] [127.0.0.1 - - [01/Jan/2022 20:41:55] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:41:57,816 INFO Thread-3 __init__.py framework_error 77] [50532] [No user_loader has been installed for this LoginManager. Refer tohttps://flask-login.readthedocs.io/en/latest/#how-it-works for more info.]
[2022-01-01 20:41:57,817 INFO Thread-3 _internal.py _log 225] [50532] [127.0.0.1 - - [01/Jan/2022 20:41:57] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 20:43:17,386 INFO MainThread _internal.py _log 225] [47968] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:43:17,498 INFO MainThread _internal.py _log 225] [50336] [ * Restarting with stat]
[2022-01-01 20:43:20,568 INFO MainThread cron_utils.py start_cron 213] [48212] [开始定时任务]
[2022-01-01 20:43:20,645 INFO MainThread base.py start 166] [48212] [Scheduler started]
[2022-01-01 20:43:20,678 WARNING MainThread _internal.py _log 225] [48212] [ * Debugger is active!]
[2022-01-01 20:43:20,707 INFO MainThread _internal.py _log 225] [48212] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:43:20,986 INFO Thread-1 _internal.py _log 225] [39836] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:44:11,738 INFO MainThread _internal.py _log 225] [48212] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:44:11,848 INFO MainThread _internal.py _log 225] [50336] [ * Restarting with stat]
[2022-01-01 20:45:18,886 INFO MainThread cron_utils.py start_cron 213] [38992] [开始定时任务]
[2022-01-01 20:45:18,979 INFO MainThread base.py start 166] [38992] [Scheduler started]
[2022-01-01 20:45:19,013 INFO MainThread _internal.py _log 225] [38992] [ * Restarting with stat]
[2022-01-01 20:45:21,714 INFO MainThread cron_utils.py start_cron 213] [50800] [开始定时任务]
[2022-01-01 20:45:21,791 INFO MainThread base.py start 166] [50800] [Scheduler started]
[2022-01-01 20:45:21,823 WARNING MainThread _internal.py _log 225] [50800] [ * Debugger is active!]
[2022-01-01 20:45:21,852 INFO MainThread _internal.py _log 225] [50800] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:45:22,121 INFO Thread-1 _internal.py _log 225] [7068] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:45:25,733 INFO Thread-2 user_controller.py post 74] [46488] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:45:25,785 INFO Thread-2 user_controller.py query_user 28] [46488] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:45:25,824 INFO Thread-2 _internal.py _log 225] [46488] [127.0.0.1 - - [01/Jan/2022 20:45:25] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:46:28,872 INFO MainThread _internal.py _log 225] [50800] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:46:29,063 INFO MainThread _internal.py _log 225] [38992] [ * Restarting with stat]
[2022-01-01 20:46:34,860 INFO MainThread cron_utils.py start_cron 213] [42116] [开始定时任务]
[2022-01-01 20:46:34,940 INFO MainThread base.py start 166] [42116] [Scheduler started]
[2022-01-01 20:46:34,998 WARNING MainThread _internal.py _log 225] [42116] [ * Debugger is active!]
[2022-01-01 20:46:35,050 INFO MainThread _internal.py _log 225] [42116] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:46:35,657 INFO Thread-1 _internal.py _log 225] [49032] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:46:47,620 INFO MainThread _internal.py _log 225] [42116] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:46:47,787 INFO MainThread _internal.py _log 225] [38992] [ * Restarting with stat]
[2022-01-01 20:47:57,306 INFO MainThread cron_utils.py start_cron 213] [36932] [开始定时任务]
[2022-01-01 20:47:57,390 INFO MainThread base.py start 166] [36932] [Scheduler started]
[2022-01-01 20:47:57,447 INFO MainThread _internal.py _log 225] [36932] [ * Restarting with stat]
[2022-01-01 20:48:02,255 INFO MainThread cron_utils.py start_cron 213] [46820] [开始定时任务]
[2022-01-01 20:48:02,348 INFO MainThread base.py start 166] [46820] [Scheduler started]
[2022-01-01 20:48:02,396 WARNING MainThread _internal.py _log 225] [46820] [ * Debugger is active!]
[2022-01-01 20:48:02,438 INFO MainThread _internal.py _log 225] [46820] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:48:02,881 INFO Thread-1 _internal.py _log 225] [44744] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:48:13,597 INFO Thread-2 user_controller.py post 73] [48972] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:48:13,630 INFO Thread-2 user_controller.py query_user 28] [48972] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:48:13,655 INFO Thread-2 _internal.py _log 225] [48972] [127.0.0.1 - - [01/Jan/2022 20:48:13] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:48:17,226 INFO Thread-3 __init__.py framework_error 77] [50660] [No user_loader has been installed for this LoginManager. Refer tohttps://flask-login.readthedocs.io/en/latest/#how-it-works for more info.]
[2022-01-01 20:48:17,226 INFO Thread-3 _internal.py _log 225] [50660] [127.0.0.1 - - [01/Jan/2022 20:48:17] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:48:53,082 INFO MainThread _internal.py _log 225] [46820] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:48:53,283 INFO MainThread _internal.py _log 225] [36932] [ * Restarting with stat]
[2022-01-01 20:52:01,878 INFO MainThread cron_utils.py start_cron 213] [48760] [开始定时任务]
[2022-01-01 20:52:01,967 INFO MainThread base.py start 166] [48760] [Scheduler started]
[2022-01-01 20:52:02,014 INFO MainThread _internal.py _log 225] [48760] [ * Restarting with stat]
[2022-01-01 20:52:05,995 INFO MainThread cron_utils.py start_cron 213] [49104] [开始定时任务]
[2022-01-01 20:52:06,081 INFO MainThread base.py start 166] [49104] [Scheduler started]
[2022-01-01 20:52:06,124 WARNING MainThread _internal.py _log 225] [49104] [ * Debugger is active!]
[2022-01-01 20:52:06,162 INFO MainThread _internal.py _log 225] [49104] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:52:06,530 INFO Thread-1 _internal.py _log 225] [48540] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:52:10,675 INFO Thread-2 user_controller.py post 84] [29844] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:52:10,741 INFO Thread-2 user_controller.py query_user 37] [29844] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:10,774 INFO Thread-2 _internal.py _log 225] [29844] [127.0.0.1 - - [01/Jan/2022 20:52:10] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:52:14,060 INFO Thread-3 user_controller.py query_user 37] [608] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:14,076 INFO Thread-3 _internal.py _log 225] [608] [127.0.0.1 - - [01/Jan/2022 20:52:14] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:52:23,365 INFO Thread-4 _internal.py _log 225] [35168] [127.0.0.1 - - [01/Jan/2022 20:52:23] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:52:26,252 INFO Thread-5 user_controller.py post 84] [47984] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:52:26,260 INFO Thread-5 user_controller.py query_user 37] [47984] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:26,286 INFO Thread-5 _internal.py _log 225] [47984] [127.0.0.1 - - [01/Jan/2022 20:52:26] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:52:29,342 INFO Thread-6 user_controller.py query_user 37] [47324] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:29,354 INFO Thread-6 _internal.py _log 225] [47324] [127.0.0.1 - - [01/Jan/2022 20:52:29] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:52:37,682 INFO Thread-7 user_controller.py post 84] [47644] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:52:37,693 INFO Thread-7 user_controller.py query_user 37] [47644] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:37,717 INFO Thread-7 _internal.py _log 225] [47644] [127.0.0.1 - - [01/Jan/2022 20:52:37] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:52:47,976 INFO Thread-8 user_controller.py query_user 37] [39628] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:47,991 INFO Thread-8 _internal.py _log 225] [39628] [127.0.0.1 - - [01/Jan/2022 20:52:47] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:52:56,813 INFO Thread-9 user_controller.py post 84] [41712] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 20:52:56,826 INFO Thread-9 user_controller.py query_user 37] [41712] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:52:56,853 INFO Thread-9 _internal.py _log 225] [41712] [127.0.0.1 - - [01/Jan/2022 20:52:56] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 20:53:03,841 INFO Thread-10 user_controller.py query_user 37] [47196] [当前登陆的用户为：<UserModel> '13800000002']
[2022-01-01 20:53:03,855 INFO Thread-10 _internal.py _log 225] [47196] [127.0.0.1 - - [01/Jan/2022 20:53:03] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 20:57:17,020 INFO MainThread _internal.py _log 225] [49104] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:57:17,172 INFO MainThread _internal.py _log 225] [48760] [ * Restarting with stat]
[2022-01-01 20:57:21,366 INFO MainThread cron_utils.py start_cron 213] [9864] [开始定时任务]
[2022-01-01 20:57:21,446 INFO MainThread base.py start 166] [9864] [Scheduler started]
[2022-01-01 20:57:21,490 WARNING MainThread _internal.py _log 225] [9864] [ * Debugger is active!]
[2022-01-01 20:57:21,531 INFO MainThread _internal.py _log 225] [9864] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:57:21,948 INFO Thread-1 _internal.py _log 225] [29844] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:58:11,636 INFO MainThread _internal.py _log 225] [9864] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:58:11,780 INFO MainThread _internal.py _log 225] [48760] [ * Restarting with stat]
[2022-01-01 20:58:15,921 INFO MainThread cron_utils.py start_cron 213] [51188] [开始定时任务]
[2022-01-01 20:58:15,998 INFO MainThread base.py start 166] [51188] [Scheduler started]
[2022-01-01 20:58:16,052 WARNING MainThread _internal.py _log 225] [51188] [ * Debugger is active!]
[2022-01-01 20:58:16,099 INFO MainThread _internal.py _log 225] [51188] [ * Debugger PIN: 133-332-853]
[2022-01-01 20:58:16,555 INFO Thread-1 _internal.py _log 225] [46900] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 20:58:48,887 INFO MainThread _internal.py _log 225] [51188] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 20:58:49,052 INFO MainThread _internal.py _log 225] [48760] [ * Restarting with stat]
[2022-01-01 21:08:10,335 INFO MainThread cron_utils.py start_cron 213] [49184] [开始定时任务]
[2022-01-01 21:08:10,425 INFO MainThread base.py start 166] [49184] [Scheduler started]
[2022-01-01 21:08:10,472 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:08:14,350 INFO MainThread cron_utils.py start_cron 213] [46236] [开始定时任务]
[2022-01-01 21:08:14,430 INFO MainThread base.py start 166] [46236] [Scheduler started]
[2022-01-01 21:08:14,472 WARNING MainThread _internal.py _log 225] [46236] [ * Debugger is active!]
[2022-01-01 21:08:14,510 INFO MainThread _internal.py _log 225] [46236] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:08:14,898 INFO Thread-1 _internal.py _log 225] [46796] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:08:54,359 INFO Thread-2 __init__.py framework_error 77] [41684] [(pymysql.err.IntegrityError) (1062, "Duplicate entry '1' for key 'PRIMARY'")
[SQL: INSERT INTO users (id, username, password, nick_name, role_id, `isDeleted`, status, create_time, update_time) VALUES (%(id)s, %(username)s, %(password)s, %(nick_name)s, %(role_id)s, %(isDeleted)s, %(status)s, %(create_time)s, %(update_time)s)]
[parameters: {'id': '1', 'username': '13800138008', 'password': '123456', 'nick_name': '荣耀', 'role_id': None, 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2022, 1, 1, 21, 8, 12, 633446), 'update_time': None}]
(Background on this error at: https://sqlalche.me/e/14/gkpj)]
[2022-01-01 21:08:54,367 INFO Thread-2 _internal.py _log 225] [41684] [127.0.0.1 - - [01/Jan/2022 21:08:54] "POST /api/user HTTP/1.1" 200 -]
[2022-01-01 21:09:16,552 INFO Thread-3 _internal.py _log 225] [15008] [127.0.0.1 - - [01/Jan/2022 21:09:16] "PUT /api/user HTTP/1.1" 200 -]
[2022-01-01 21:10:06,180 INFO Thread-4 _internal.py _log 225] [48440] [127.0.0.1 - - [01/Jan/2022 21:10:06] "DELETE /api/user HTTP/1.1" 200 -]
[2022-01-01 21:10:20,899 INFO Thread-5 _internal.py _log 225] [47272] [127.0.0.1 - - [01/Jan/2022 21:10:20] "DELETE /api/user HTTP/1.1" 200 -]
[2022-01-01 21:13:29,713 INFO Thread-6 _internal.py _log 225] [46672] [127.0.0.1 - - [01/Jan/2022 21:13:29] "POST /api/project HTTP/1.1" 200 -]
[2022-01-01 21:13:32,004 INFO Thread-7 __init__.py abort_401 45] [36292] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:13:32,004 INFO Thread-7 _internal.py _log 225] [36292] [127.0.0.1 - - [01/Jan/2022 21:13:32] "GET /api/project?project_name=拉勾&type=search HTTP/1.1" 200 -]
[2022-01-01 21:13:34,586 INFO Thread-8 __init__.py abort_401 45] [47324] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:13:34,588 INFO Thread-8 _internal.py _log 225] [47324] [127.0.0.1 - - [01/Jan/2022 21:13:34] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 21:13:45,134 INFO Thread-9 __init__.py abort_401 45] [9708] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:13:45,135 INFO Thread-9 _internal.py _log 225] [9708] [127.0.0.1 - - [01/Jan/2022 21:13:45] "GET /api/project?type=query_list&page=1&size=10 HTTP/1.1" 200 -]
[2022-01-01 21:14:14,715 INFO MainThread _internal.py _log 225] [46236] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 21:14:14,833 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:14:17,722 INFO MainThread cron_utils.py start_cron 213] [27724] [开始定时任务]
[2022-01-01 21:14:17,815 INFO MainThread base.py start 166] [27724] [Scheduler started]
[2022-01-01 21:14:17,852 WARNING MainThread _internal.py _log 225] [27724] [ * Debugger is active!]
[2022-01-01 21:14:17,885 INFO MainThread _internal.py _log 225] [27724] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:14:18,192 INFO Thread-1 _internal.py _log 225] [44780] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:14:35,930 INFO Thread-2 _internal.py _log 225] [48868] [127.0.0.1 - - [01/Jan/2022 21:14:35] "POST /api/project HTTP/1.1" 200 -]
[2022-01-01 21:14:38,440 INFO Thread-3 __init__.py abort_401 45] [48624] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:14:38,441 INFO Thread-3 _internal.py _log 225] [48624] [127.0.0.1 - - [01/Jan/2022 21:14:38] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 21:14:44,814 INFO Thread-4 _internal.py _log 225] [21264] [127.0.0.1 - - [01/Jan/2022 21:14:44] "POST /api/project HTTP/1.1" 200 -]
[2022-01-01 21:14:46,835 INFO Thread-5 __init__.py abort_401 45] [49188] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:14:46,836 INFO Thread-5 _internal.py _log 225] [49188] [127.0.0.1 - - [01/Jan/2022 21:14:46] "GET /api/project?project_name=拉勾&type=search HTTP/1.1" 200 -]
[2022-01-01 21:15:55,097 INFO MainThread _internal.py _log 225] [27724] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 21:15:55,215 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:15:57,996 INFO MainThread cron_utils.py start_cron 213] [49324] [开始定时任务]
[2022-01-01 21:15:58,075 INFO MainThread base.py start 166] [49324] [Scheduler started]
[2022-01-01 21:15:58,108 WARNING MainThread _internal.py _log 225] [49324] [ * Debugger is active!]
[2022-01-01 21:15:58,139 INFO MainThread _internal.py _log 225] [49324] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:15:58,507 INFO Thread-1 _internal.py _log 225] [45100] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:16:03,537 INFO Thread-2 test_suite_controller.py query_suite_by_name 50] [39400] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:16:03,538 INFO Thread-2 test_suite_controller.py query_suite_by_name 59] [39400] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:16:03,550 INFO Thread-2 _internal.py _log 225] [39400] [127.0.0.1 - - [01/Jan/2022 21:16:03] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:16:06,208 INFO Thread-3 test_suite_controller.py query_suite_by_id 37] [23476] [查询的测试套件id为：1的详情数据为：<TestSuiteModel> '登陆2']
[2022-01-01 21:16:06,209 INFO Thread-3 test_suite_controller.py query_suite_by_id 40] [23476] [查询的测试套件id为：1的详情数据转化为json后：{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]
[2022-01-01 21:16:06,209 INFO Thread-3 test_suite_controller.py get 213] [23476] [把日期对象转化为字符串之后，的结果为：{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}]
[2022-01-01 21:16:06,222 INFO Thread-3 _internal.py _log 225] [23476] [127.0.0.1 - - [01/Jan/2022 21:16:06] "GET /api/suite?project_id=1&id=1&type=query_detail HTTP/1.1" 200 -]
[2022-01-01 21:16:10,301 INFO Thread-4 _internal.py _log 225] [49888] [127.0.0.1 - - [01/Jan/2022 21:16:10] "GET /api/project?project_name=拉勾&type=search HTTP/1.1" 200 -]
[2022-01-01 21:16:12,564 INFO Thread-5 _internal.py _log 225] [43940] [127.0.0.1 - - [01/Jan/2022 21:16:12] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 21:16:26,146 INFO MainThread _internal.py _log 225] [49324] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 21:16:26,297 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:16:29,998 INFO MainThread cron_utils.py start_cron 213] [47884] [开始定时任务]
[2022-01-01 21:16:30,079 INFO MainThread base.py start 166] [47884] [Scheduler started]
[2022-01-01 21:16:30,114 WARNING MainThread _internal.py _log 225] [47884] [ * Debugger is active!]
[2022-01-01 21:16:30,145 INFO MainThread _internal.py _log 225] [47884] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:16:30,433 INFO Thread-1 _internal.py _log 225] [41712] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:16:39,544 INFO MainThread _internal.py _log 225] [47884] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\user_controller.py', reloading]
[2022-01-01 21:16:39,640 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:16:42,804 INFO MainThread cron_utils.py start_cron 213] [50576] [开始定时任务]
[2022-01-01 21:16:42,896 INFO MainThread base.py start 166] [50576] [Scheduler started]
[2022-01-01 21:16:42,933 WARNING MainThread _internal.py _log 225] [50576] [ * Debugger is active!]
[2022-01-01 21:16:42,967 INFO MainThread _internal.py _log 225] [50576] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:16:43,357 INFO Thread-1 _internal.py _log 225] [43328] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:19:37,746 INFO Thread-2 test_suite_controller.py query_suite_by_id 37] [34668] [查询的测试套件id为：1的详情数据为：<TestSuiteModel> '登陆2']
[2022-01-01 21:19:37,747 INFO Thread-2 test_suite_controller.py query_suite_by_id 40] [34668] [查询的测试套件id为：1的详情数据转化为json后：{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 26, 17, 46, 3), 'update_time': None, 'operation': None}]
[2022-01-01 21:19:37,748 INFO Thread-2 test_suite_controller.py get 213] [34668] [把日期对象转化为字符串之后，的结果为：{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}]
[2022-01-01 21:19:37,759 INFO Thread-2 _internal.py _log 225] [34668] [127.0.0.1 - - [01/Jan/2022 21:19:37] "GET /api/suite?project_id=1&id=1&type=query_detail HTTP/1.1" 200 -]
[2022-01-01 21:19:40,012 INFO Thread-3 test_suite_controller.py query_suite_by_name 50] [30124] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:19:40,012 INFO Thread-3 test_suite_controller.py query_suite_by_name 59] [30124] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:19:40,024 INFO Thread-3 _internal.py _log 225] [30124] [127.0.0.1 - - [01/Jan/2022 21:19:40] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:19:43,115 INFO Thread-4 test_suite_controller.py query_list 69] [49724] [查询出的测试套件列表数据为：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2', <TestSuiteModel> '支付']]
[2022-01-01 21:19:43,116 INFO Thread-4 test_suite_controller.py query_list 77] [49724] [查询出的测试套件列表数据并转化为json为：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2', <TestSuiteModel> '支付']]
[2022-01-01 21:19:43,129 INFO Thread-4 _internal.py _log 225] [49724] [127.0.0.1 - - [01/Jan/2022 21:19:43] "GET /api/suite?project_id=1&type=query_list&page=1&size=10 HTTP/1.1" 200 -]
[2022-01-01 21:19:48,328 INFO Thread-5 user_controller.py post 118] [50140] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800000002",\r\n    "password":"123456"\r\n}']
[2022-01-01 21:19:48,345 INFO Thread-5 user_controller.py query_user 36] [50140] [当前登陆的用户为：None]
[2022-01-01 21:19:48,358 INFO Thread-5 _internal.py _log 225] [50140] [127.0.0.1 - - [01/Jan/2022 21:19:48] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 21:19:50,918 INFO Thread-6 test_suite_controller.py query_suite_by_name 50] [48288] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:19:50,918 INFO Thread-6 test_suite_controller.py query_suite_by_name 59] [48288] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:19:50,930 INFO Thread-6 _internal.py _log 225] [48288] [127.0.0.1 - - [01/Jan/2022 21:19:50] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:19:56,803 INFO Thread-7 test_suite_controller.py query_suite_by_name 50] [45496] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:19:56,804 INFO Thread-7 test_suite_controller.py query_suite_by_name 59] [45496] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:19:56,818 INFO Thread-7 _internal.py _log 225] [45496] [127.0.0.1 - - [01/Jan/2022 21:19:56] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:20:34,216 INFO MainThread _internal.py _log 225] [50576] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 21:20:34,331 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:20:37,751 INFO MainThread cron_utils.py start_cron 213] [49708] [开始定时任务]
[2022-01-01 21:20:37,859 INFO MainThread base.py start 166] [49708] [Scheduler started]
[2022-01-01 21:20:37,906 WARNING MainThread _internal.py _log 225] [49708] [ * Debugger is active!]
[2022-01-01 21:20:37,944 INFO MainThread _internal.py _log 225] [49708] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:20:38,331 INFO Thread-1 _internal.py _log 225] [31676] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:21:15,026 INFO MainThread _internal.py _log 225] [49708] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 21:21:15,174 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:21:19,919 INFO MainThread cron_utils.py start_cron 213] [49796] [开始定时任务]
[2022-01-01 21:21:20,006 INFO MainThread base.py start 166] [49796] [Scheduler started]
[2022-01-01 21:21:20,051 WARNING MainThread _internal.py _log 225] [49796] [ * Debugger is active!]
[2022-01-01 21:21:20,092 INFO MainThread _internal.py _log 225] [49796] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:21:20,506 INFO Thread-1 _internal.py _log 225] [46324] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:22:08,190 INFO MainThread _internal.py _log 225] [49796] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 21:22:08,325 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:22:12,240 INFO MainThread cron_utils.py start_cron 213] [28036] [开始定时任务]
[2022-01-01 21:22:12,319 INFO MainThread base.py start 166] [28036] [Scheduler started]
[2022-01-01 21:22:12,365 WARNING MainThread _internal.py _log 225] [28036] [ * Debugger is active!]
[2022-01-01 21:22:12,407 INFO MainThread _internal.py _log 225] [28036] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:22:12,805 INFO Thread-1 _internal.py _log 225] [46840] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:22:17,164 INFO MainThread _internal.py _log 225] [28036] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_project_controller.py', reloading]
[2022-01-01 21:22:17,327 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:22:21,894 INFO MainThread cron_utils.py start_cron 213] [43556] [开始定时任务]
[2022-01-01 21:22:21,976 INFO MainThread base.py start 166] [43556] [Scheduler started]
[2022-01-01 21:22:22,054 WARNING MainThread _internal.py _log 225] [43556] [ * Debugger is active!]
[2022-01-01 21:22:22,112 INFO MainThread _internal.py _log 225] [43556] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:22:22,623 INFO Thread-1 _internal.py _log 225] [43328] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:22:26,380 INFO Thread-2 __init__.py abort_401 45] [31904] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:26,382 INFO Thread-2 _internal.py _log 225] [31904] [127.0.0.1 - - [01/Jan/2022 21:22:26] "POST /api/project HTTP/1.1" 200 -]
[2022-01-01 21:22:31,522 INFO Thread-3 __init__.py abort_401 45] [37184] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:31,523 INFO Thread-3 _internal.py _log 225] [37184] [127.0.0.1 - - [01/Jan/2022 21:22:31] "GET /api/project?id=1&type=query_detail&project_name=拉勾教育 HTTP/1.1" 200 -]
[2022-01-01 21:22:33,979 INFO Thread-4 __init__.py abort_401 45] [7244] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:33,980 INFO Thread-4 _internal.py _log 225] [7244] [127.0.0.1 - - [01/Jan/2022 21:22:33] "GET /api/project?project_name=拉勾&type=search HTTP/1.1" 200 -]
[2022-01-01 21:22:36,239 INFO Thread-5 __init__.py abort_401 45] [30572] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:36,240 INFO Thread-5 _internal.py _log 225] [30572] [127.0.0.1 - - [01/Jan/2022 21:22:36] "GET /api/project?type=query_list&page=1&size=10 HTTP/1.1" 200 -]
[2022-01-01 21:22:39,439 INFO Thread-6 __init__.py abort_401 45] [50768] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:39,447 INFO Thread-6 _internal.py _log 225] [50768] [127.0.0.1 - - [01/Jan/2022 21:22:39] "PUT /api/project HTTP/1.1" 200 -]
[2022-01-01 21:22:42,153 INFO Thread-7 __init__.py abort_401 45] [51148] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:22:42,154 INFO Thread-7 _internal.py _log 225] [51148] [127.0.0.1 - - [01/Jan/2022 21:22:42] "DELETE /api/project HTTP/1.1" 200 -]
[2022-01-01 21:22:50,056 INFO Thread-8 _internal.py _log 225] [43912] [127.0.0.1 - - [01/Jan/2022 21:22:50] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:23:08,731 INFO MainThread _internal.py _log 225] [43556] [ * Detected change in 'C:\\Users\\wind\\PycharmProjects\\taitan_interface_platform\\backend\\controller\\test_suite_controller.py', reloading]
[2022-01-01 21:23:08,959 INFO MainThread _internal.py _log 225] [49184] [ * Restarting with stat]
[2022-01-01 21:25:29,903 INFO MainThread cron_utils.py start_cron 213] [44304] [开始定时任务]
[2022-01-01 21:25:29,984 INFO MainThread base.py start 166] [44304] [Scheduler started]
[2022-01-01 21:25:30,039 INFO MainThread _internal.py _log 225] [44304] [ * Restarting with stat]
[2022-01-01 21:25:33,839 INFO MainThread cron_utils.py start_cron 213] [50508] [开始定时任务]
[2022-01-01 21:25:33,915 INFO MainThread base.py start 166] [50508] [Scheduler started]
[2022-01-01 21:25:33,968 WARNING MainThread _internal.py _log 225] [50508] [ * Debugger is active!]
[2022-01-01 21:25:34,009 INFO MainThread _internal.py _log 225] [50508] [ * Debugger PIN: 133-332-853]
[2022-01-01 21:25:34,403 INFO Thread-1 _internal.py _log 225] [5412] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
[2022-01-01 21:25:39,660 INFO Thread-2 __init__.py abort_401 45] [49724] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:39,662 INFO Thread-2 _internal.py _log 225] [49724] [127.0.0.1 - - [01/Jan/2022 21:25:39] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:25:41,963 INFO Thread-3 __init__.py abort_401 45] [22640] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:41,964 INFO Thread-3 _internal.py _log 225] [22640] [127.0.0.1 - - [01/Jan/2022 21:25:41] "GET /api/suite?project_id=1&id=1&type=query_detail HTTP/1.1" 200 -]
[2022-01-01 21:25:44,090 INFO Thread-4 __init__.py abort_401 45] [50884] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:44,091 INFO Thread-4 _internal.py _log 225] [50884] [127.0.0.1 - - [01/Jan/2022 21:25:44] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:25:46,251 INFO Thread-5 __init__.py abort_401 45] [47004] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:46,253 INFO Thread-5 _internal.py _log 225] [47004] [127.0.0.1 - - [01/Jan/2022 21:25:46] "GET /api/suite?project_id=1&type=query_list&page=1&size=10 HTTP/1.1" 200 -]
[2022-01-01 21:25:48,901 INFO Thread-6 __init__.py abort_401 45] [30904] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:48,902 INFO Thread-6 _internal.py _log 225] [30904] [127.0.0.1 - - [01/Jan/2022 21:25:48] "PUT /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:25:52,709 INFO Thread-7 __init__.py abort_401 45] [43876] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:52,710 INFO Thread-7 _internal.py _log 225] [43876] [127.0.0.1 - - [01/Jan/2022 21:25:52] "DELETE /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:25:55,735 INFO Thread-8 __init__.py abort_401 45] [47692] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:55,736 INFO Thread-8 _internal.py _log 225] [47692] [127.0.0.1 - - [01/Jan/2022 21:25:55] "POST /api/run/testcase HTTP/1.1" 200 -]
[2022-01-01 21:25:58,221 INFO Thread-9 __init__.py abort_401 45] [14080] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:25:58,222 INFO Thread-9 _internal.py _log 225] [14080] [127.0.0.1 - - [01/Jan/2022 21:25:58] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:26:00,651 INFO Thread-10 __init__.py abort_401 45] [45436] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:00,653 INFO Thread-10 _internal.py _log 225] [45436] [127.0.0.1 - - [01/Jan/2022 21:26:00] "POST /api/suite HTTP/1.1" 200 -]
[2022-01-01 21:26:03,904 INFO Thread-11 __init__.py abort_401 45] [12132] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:03,905 INFO Thread-11 _internal.py _log 225] [12132] [127.0.0.1 - - [01/Jan/2022 21:26:03] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 21:26:06,787 INFO Thread-12 __init__.py abort_401 45] [31516] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:06,788 INFO Thread-12 _internal.py _log 225] [31516] [127.0.0.1 - - [01/Jan/2022 21:26:06] "GET /api/testcase?project_id=39&suite_id=9&id=1&type=query_detail HTTP/1.1" 200 -]
[2022-01-01 21:26:09,605 INFO Thread-13 __init__.py abort_401 45] [43464] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:09,606 INFO Thread-13 _internal.py _log 225] [43464] [127.0.0.1 - - [01/Jan/2022 21:26:09] "GET /api/testcase?suite_id=9&type=search&case_name=登录成功 HTTP/1.1" 200 -]
[2022-01-01 21:26:12,850 INFO Thread-14 __init__.py abort_401 45] [43684] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:12,851 INFO Thread-14 _internal.py _log 225] [43684] [127.0.0.1 - - [01/Jan/2022 21:26:12] "GET /api/testcase?project_id=39&suite_id=9&page=1&size=10&type=query_list HTTP/1.1" 200 -]
[2022-01-01 21:26:15,784 INFO Thread-15 __init__.py abort_401 45] [48184] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:15,785 INFO Thread-15 _internal.py _log 225] [48184] [127.0.0.1 - - [01/Jan/2022 21:26:15] "PUT /api/testcase HTTP/1.1" 200 -]
[2022-01-01 21:26:18,768 INFO Thread-16 __init__.py abort_401 45] [48896] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:18,769 INFO Thread-16 _internal.py _log 225] [48896] [127.0.0.1 - - [01/Jan/2022 21:26:18] "DELETE /api/testcase HTTP/1.1" 200 -]
[2022-01-01 21:26:21,021 INFO Thread-17 __init__.py abort_401 45] [49088] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:21,022 INFO Thread-17 _internal.py _log 225] [49088] [127.0.0.1 - - [01/Jan/2022 21:26:21] "POST /api/run/testcase HTTP/1.1" 200 -]
[2022-01-01 21:26:23,168 INFO Thread-18 __init__.py abort_401 45] [26800] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:26:23,168 INFO Thread-18 _internal.py _log 225] [26800] [127.0.0.1 - - [01/Jan/2022 21:26:23] "POST /api/testcase HTTP/1.1" 200 -]
[2022-01-01 21:26:36,037 INFO Thread-19 user_controller.py post 120] [7352] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138008",\r\n    "password":"123456"\r\n}']
[2022-01-01 21:26:36,083 INFO Thread-19 user_controller.py query_user 36] [7352] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:26:36,114 INFO Thread-19 _internal.py _log 225] [7352] [127.0.0.1 - - [01/Jan/2022 21:26:36] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 21:26:40,559 INFO Thread-20 user_controller.py query_user 36] [47280] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:26:40,571 INFO Thread-20 test_case_controller.py query_testcase_by_name 95] [47280] [该用例对应的测试套件id为{suite_id}的数据不存在]
[2022-01-01 21:26:40,582 INFO Thread-20 _internal.py _log 225] [47280] [127.0.0.1 - - [01/Jan/2022 21:26:40] "GET /api/testcase?suite_id=9&type=search&case_name=登录成功 HTTP/1.1" 200 -]
[2022-01-01 21:26:43,513 INFO Thread-21 user_controller.py query_user 36] [41168] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:26:43,521 INFO Thread-21 test_case_controller.py query_list 122] [41168] [该用例对应的测试套件id为{suite_id}的数据不存在]
[2022-01-01 21:26:43,532 INFO Thread-21 _internal.py _log 225] [41168] [127.0.0.1 - - [01/Jan/2022 21:26:43] "GET /api/testcase?project_id=39&suite_id=9&page=1&size=10&type=query_list HTTP/1.1" 200 -]
[2022-01-01 21:27:21,881 INFO Thread-22 user_controller.py query_user 36] [47060] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:27:21,930 INFO Thread-22 test_case_controller.py query_testase_by_id 77] [47060] [查询的测试用例id为：4的详情数据为：<TestCaseModel> '用例4']
[2022-01-01 21:27:21,932 INFO Thread-22 test_case_controller.py query_testase_by_id 83] [47060] [查询的测试用例id为：4的详情数据转化为json后：{'id': 4, 'case_name': '用例4', 'method': 'POST', 'protocal': 'http', 'host': '10.1.201.231', 'port': 8080, 'path': '/ssm_web/user/login', 'params': "{'phone': '{{phone}}', 'password': '{{password}}'}", 'headers': "{'Content-Type': 'application/json'}", 'body': '{}', 'description': '拉勾商城登录成功', 'predict': '\x7f{‘a’：‘b’}', 'suite_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': datetime.datetime(2021, 12, 31, 14, 8, 1), 'update_time': None, 'operation': None}]
[2022-01-01 21:27:21,945 INFO Thread-22 _internal.py _log 225] [47060] [127.0.0.1 - - [01/Jan/2022 21:27:21] "GET /api/testcase?project_id=1&suite_id=1&id=4&type=query_detail HTTP/1.1" 200 -]
[2022-01-01 21:27:26,962 INFO Thread-23 user_controller.py query_user 36] [9708] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:27:26,979 INFO Thread-23 test_suite_controller.py query_suite_by_name 52] [9708] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:27:26,980 INFO Thread-23 test_suite_controller.py query_suite_by_name 61] [9708] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}, {'id': 4, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 21:22:20', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:27:26,993 INFO Thread-23 _internal.py _log 225] [9708] [127.0.0.1 - - [01/Jan/2022 21:27:26] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:27:34,032 INFO Thread-24 user_controller.py query_user 36] [46488] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:27:34,045 INFO Thread-24 _internal.py _log 225] [46488] [127.0.0.1 - - [01/Jan/2022 21:27:34] "POST /api/logout HTTP/1.1" 200 -]
[2022-01-01 21:27:37,406 INFO Thread-25 __init__.py abort_401 45] [47968] [401 Unauthorized: The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required.]
[2022-01-01 21:27:37,407 INFO Thread-25 _internal.py _log 225] [47968] [127.0.0.1 - - [01/Jan/2022 21:27:37] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-01 21:27:41,058 INFO Thread-26 user_controller.py post 120] [47536] [给登陆接口传入的请求数据为：b'{\r\n    "operation": "login",\r\n    "username":"13800138008",\r\n    "password":"123456"\r\n}']
[2022-01-01 21:27:41,068 INFO Thread-26 user_controller.py query_user 36] [47536] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:27:41,090 INFO Thread-26 _internal.py _log 225] [47536] [127.0.0.1 - - [01/Jan/2022 21:27:41] "POST /api/login HTTP/1.1" 200 -]
[2022-01-01 21:27:44,927 INFO Thread-27 user_controller.py query_user 36] [32904] [当前登陆的用户为：<UserModel> '13800138008']
[2022-01-01 21:27:44,937 INFO Thread-27 test_suite_controller.py query_suite_by_name 52] [32904] [根据测试套件名称 [登陆] 搜索出来的数据有：[<TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2', <TestSuiteModel> '登陆2']]
[2022-01-01 21:27:44,938 INFO Thread-27 test_suite_controller.py query_suite_by_name 61] [32904] [根据测试套件名称 [登陆] 搜索出来的数据并转化为json后：[{'id': 1, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2021-12-26 17:46:03', 'update_time': None, 'operation': None}, {'id': 2, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 16:33:46', 'update_time': None, 'operation': 'add'}, {'id': 4, 'suite_name': '登陆2', 'description': '描述内容', 'project_id': 1, 'isDeleted': '0', 'status': '1', 'create_time': '2022-01-01 21:22:20', 'update_time': None, 'operation': 'add'}]]
[2022-01-01 21:27:44,948 INFO Thread-27 _internal.py _log 225] [32904] [127.0.0.1 - - [01/Jan/2022 21:27:44] "GET /api/suite?project_id=1&type=search&suite_name=登陆 HTTP/1.1" 200 -]
[2022-01-02 10:39:35,351 INFO MainThread cron_utils.py start_cron 213] [56676] [开始定时任务]
[2022-01-02 10:39:35,450 INFO MainThread base.py start 166] [56676] [Scheduler started]
[2022-01-02 10:39:35,587 INFO MainThread _internal.py _log 225] [56676] [ * Restarting with stat]
[2022-01-02 10:39:39,336 INFO MainThread cron_utils.py start_cron 213] [53360] [开始定时任务]
[2022-01-02 10:39:39,415 INFO MainThread base.py start 166] [53360] [Scheduler started]
[2022-01-02 10:39:39,454 WARNING MainThread _internal.py _log 225] [53360] [ * Debugger is active!]
[2022-01-02 10:39:39,492 INFO MainThread _internal.py _log 225] [53360] [ * Debugger PIN: 133-332-853]
[2022-01-02 10:39:39,850 INFO Thread-1 _internal.py _log 225] [51968] [ * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)]
